wandb: Starting wandb agent üïµÔ∏è
wandb: Starting wandb agent üïµÔ∏è
wandb: Starting wandb agent üïµÔ∏è
wandb: Starting wandb agent üïµÔ∏è
wandb: Starting wandb agent üïµÔ∏è
wandb: Starting wandb agent üïµÔ∏è
2023-08-23 22:57:28,679 - wandb.wandb_agent - INFO - Running runs: []
2023-08-23 15:57:28,762 - wandb.wandb_agent - INFO - Running runs: []
2023-08-23 22:57:28,824 - wandb.wandb_agent - INFO - Running runs: []
2023-08-23 15:57:28,879 - wandb.wandb_agent - INFO - Running runs: []
2023-08-23 22:57:28,996 - wandb.wandb_agent - INFO - Agent received command: run
2023-08-23 22:57:28,996 - wandb.wandb_agent - INFO - Agent starting run with config:
	gamma: 0.099
	lr: 0.001
2023-08-23 22:57:28,998 - wandb.wandb_agent - INFO - About to run command: /usr/bin/env python /nas/ucb/mason/ethically-compliant-rl/CPO/train_cpo.py --gamma=0.099 --lr=0.001
2023-08-23 15:57:29,025 - wandb.wandb_agent - INFO - Running runs: []
2023-08-23 15:57:29,065 - wandb.wandb_agent - INFO - Agent received command: run
2023-08-23 15:57:29,065 - wandb.wandb_agent - INFO - Agent starting run with config:
	actor_lr: 0.0005
	critic_lr: 0.0005
	gamma: 0.099
2023-08-23 15:57:29,067 - wandb.wandb_agent - INFO - About to run command: /usr/bin/env python /nas/ucb/mason/ethically-compliant-rl/CVPO/train_cvpo.py --actor_lr=0.0005 --critic_lr=0.0005 --gamma=0.099
2023-08-23 15:57:29,143 - wandb.wandb_agent - INFO - Running runs: []
2023-08-23 22:57:29,291 - wandb.wandb_agent - INFO - Agent received command: run
2023-08-23 22:57:29,292 - wandb.wandb_agent - INFO - Agent starting run with config:
	actor_lr: 0.0005
	critic_lr: 0.0005
	gamma: 0.095
2023-08-23 22:57:29,294 - wandb.wandb_agent - INFO - About to run command: /usr/bin/env python /nas/ucb/mason/ethically-compliant-rl/CVPO/train_cvpo.py --actor_lr=0.0005 --critic_lr=0.0005 --gamma=0.095
2023-08-23 15:57:29,341 - wandb.wandb_agent - INFO - Agent received command: run
2023-08-23 15:57:29,341 - wandb.wandb_agent - INFO - Agent starting run with config:
	gamma: 0.095
	lr: 0.001
2023-08-23 15:57:29,344 - wandb.wandb_agent - INFO - About to run command: /usr/bin/env python /nas/ucb/mason/ethically-compliant-rl/CPO/train_cpo.py --gamma=0.095 --lr=0.001
2023-08-23 15:57:29,545 - wandb.wandb_agent - INFO - Agent received command: run
2023-08-23 15:57:29,545 - wandb.wandb_agent - INFO - Agent starting run with config:
	actor_lr: 0.001
	critic_lr: 0.0005
	gamma: 0.095
2023-08-23 15:57:29,548 - wandb.wandb_agent - INFO - About to run command: /usr/bin/env python /nas/ucb/mason/ethically-compliant-rl/CVPO/train_cvpo.py --actor_lr=0.001 --critic_lr=0.0005 --gamma=0.095
2023-08-23 15:57:29,589 - wandb.wandb_agent - INFO - Agent received command: run
2023-08-23 15:57:29,589 - wandb.wandb_agent - INFO - Agent starting run with config:
	gamma: 0.095
	lr: 0.001
2023-08-23 15:57:29,591 - wandb.wandb_agent - INFO - About to run command: /usr/bin/env python /nas/ucb/mason/ethically-compliant-rl/CPO/train_cpo.py --gamma=0.095 --lr=0.001
wandb: Currently logged in as: mason-nakamura1. Use `wandb login --relogin` to force relogin
wandb: WARNING Ignored wandb.init() arg project when running a sweep.
wandb: WARNING Ignored wandb.init() arg entity when running a sweep.
wandb: Currently logged in as: mason-nakamura1. Use `wandb login --relogin` to force relogin
wandb: WARNING Ignored wandb.init() arg project when running a sweep.
wandb: WARNING Ignored wandb.init() arg entity when running a sweep.
wandb: wandb version 0.15.8 is available!  To upgrade, please run:
wandb:  $ pip install wandb --upgrade
wandb: Tracking run with wandb version 0.14.2
wandb: Run data is saved locally in /nas/ucb/mason/ethically-compliant-rl/wandb/run-20230823_225731-h7abijfw
wandb: Run `wandb offline` to turn off syncing.
wandb: Syncing run skilled-sweep-1
wandb: ‚≠êÔ∏è View project at https://wandb.ai/ecrl/fast-safe-rl
wandb: üßπ View sweep at https://wandb.ai/ecrl/fast-safe-rl/sweeps/tea6k681
wandb: üöÄ View run at https://wandb.ai/ecrl/fast-safe-rl/runs/h7abijfw
wandb: Currently logged in as: mason-nakamura1. Use `wandb login --relogin` to force relogin
wandb: WARNING Ignored wandb.init() arg project when running a sweep.
wandb: WARNING Ignored wandb.init() arg entity when running a sweep.
wandb: Currently logged in as: mason-nakamura1. Use `wandb login --relogin` to force relogin
wandb: WARNING Ignored wandb.init() arg project when running a sweep.
wandb: WARNING Ignored wandb.init() arg entity when running a sweep.
wandb: Currently logged in as: mason-nakamura1. Use `wandb login --relogin` to force relogin
wandb: WARNING Ignored wandb.init() arg project when running a sweep.
wandb: WARNING Ignored wandb.init() arg entity when running a sweep.
wandb: wandb version 0.15.8 is available!  To upgrade, please run:
wandb:  $ pip install wandb --upgrade
wandb: Tracking run with wandb version 0.14.2
wandb: Run data is saved locally in /nas/ucb/mason/ethically-compliant-rl/wandb/run-20230823_225731-q17z0y00
wandb: Run `wandb offline` to turn off syncing.
wandb: Syncing run hardy-sweep-2
wandb: ‚≠êÔ∏è View project at https://wandb.ai/ecrl/fast-safe-rl
wandb: üßπ View sweep at https://wandb.ai/ecrl/fast-safe-rl/sweeps/3qe1o66j
wandb: üöÄ View run at https://wandb.ai/ecrl/fast-safe-rl/runs/q17z0y00
wandb: wandb version 0.15.8 is available!  To upgrade, please run:
wandb:  $ pip install wandb --upgrade
wandb: Tracking run with wandb version 0.14.2
wandb: Run data is saved locally in /nas/ucb/mason/ethically-compliant-rl/wandb/run-20230823_155731-rybmrwkh
wandb: Run `wandb offline` to turn off syncing.
wandb: Syncing run decent-sweep-2
wandb: ‚≠êÔ∏è View project at https://wandb.ai/ecrl/fast-safe-rl
wandb: üßπ View sweep at https://wandb.ai/ecrl/fast-safe-rl/sweeps/tea6k681
wandb: üöÄ View run at https://wandb.ai/ecrl/fast-safe-rl/runs/rybmrwkh
wandb: wandb version 0.15.8 is available!  To upgrade, please run:
wandb:  $ pip install wandb --upgrade
wandb: Tracking run with wandb version 0.14.2
wandb: Run data is saved locally in /nas/ucb/mason/ethically-compliant-rl/wandb/run-20230823_155731-fktjssxy
wandb: Run `wandb offline` to turn off syncing.
wandb: Syncing run celestial-sweep-1
wandb: ‚≠êÔ∏è View project at https://wandb.ai/ecrl/fast-safe-rl
wandb: üßπ View sweep at https://wandb.ai/ecrl/fast-safe-rl/sweeps/3qe1o66j
wandb: üöÄ View run at https://wandb.ai/ecrl/fast-safe-rl/runs/fktjssxy
wandb: Currently logged in as: mason-nakamura1. Use `wandb login --relogin` to force relogin
wandb: WARNING Ignored wandb.init() arg project when running a sweep.
wandb: WARNING Ignored wandb.init() arg entity when running a sweep.
wandb: wandb version 0.15.8 is available!  To upgrade, please run:
wandb:  $ pip install wandb --upgrade
wandb: Tracking run with wandb version 0.14.2
wandb: Run data is saved locally in /nas/ucb/mason/ethically-compliant-rl/wandb/run-20230823_155732-0obizknr
wandb: Run `wandb offline` to turn off syncing.
wandb: Syncing run azure-sweep-3
wandb: ‚≠êÔ∏è View project at https://wandb.ai/ecrl/fast-safe-rl
wandb: üßπ View sweep at https://wandb.ai/ecrl/fast-safe-rl/sweeps/tea6k681
wandb: üöÄ View run at https://wandb.ai/ecrl/fast-safe-rl/runs/0obizknr
wandb: wandb version 0.15.8 is available!  To upgrade, please run:
wandb:  $ pip install wandb --upgrade
wandb: Tracking run with wandb version 0.14.2
wandb: Run data is saved locally in /nas/ucb/mason/ethically-compliant-rl/wandb/run-20230823_155732-zdual6e6
wandb: Run `wandb offline` to turn off syncing.
wandb: Syncing run hardy-sweep-3
wandb: ‚≠êÔ∏è View project at https://wandb.ai/ecrl/fast-safe-rl
wandb: üßπ View sweep at https://wandb.ai/ecrl/fast-safe-rl/sweeps/3qe1o66j
wandb: üöÄ View run at https://wandb.ai/ecrl/fast-safe-rl/runs/zdual6e6
2023-08-23 22:57:34,006 - wandb.wandb_agent - INFO - Running runs: ['h7abijfw']
2023-08-23 15:57:34,076 - wandb.wandb_agent - INFO - Running runs: ['fktjssxy']
2023-08-23 22:57:34,303 - wandb.wandb_agent - INFO - Running runs: ['q17z0y00']
2023-08-23 15:57:34,354 - wandb.wandb_agent - INFO - Running runs: ['rybmrwkh']
2023-08-23 15:57:34,557 - wandb.wandb_agent - INFO - Running runs: ['zdual6e6']
2023-08-23 15:57:34,600 - wandb.wandb_agent - INFO - Running runs: ['0obizknr']
[32;1mLogging data to logs/CVPO-sweep-700epochs/parking-v0-cost-[5.0]/cvpo_constraint_typelines_cost5.0_critic_lr0.0005_gamma0.095_step_per_epoch1000-749c/progress.txt[0m
[32;1mLogging data to logs/CPO-sweep-700epochs/parking-v0-cost-[5.0]/cpo_constraint_typelines_cost5.0_gamma0.099_step_per_epoch1000-cf02/progress.txt[0m
[32;1mLogging data to logs/CVPO-sweep-700epochs/parking-v0-cost-[5.0]/cvpo_constraint_typelines_cost5.0_critic_lr0.0005_gamma0.099_step_per_epoch1000-a013/progress.txt[0m
[32;1mLogging data to logs/CPO-sweep-700epochs/parking-v0-cost-[5.0]/cpo_constraint_typelines_cost5.0_gamma0.095_step_per_epoch1000-3621/progress.txt[0m
[32;1mLogging data to logs/CPO-sweep-700epochs/parking-v0-cost-[5.0]/cpo_constraint_typelines_cost5.0_gamma0.095_step_per_epoch1000-f17a/progress.txt[0m
[32;1mLogging data to logs/CVPO-sweep-700epochs/parking-v0-cost-[5.0]/cvpo_actor_lr0.001_constraint_typelines_cost5.0_critic_lr0.0005_gamma0.095_step_per_epoch1000-2c96/progress.txt[0m
Epoch #1:   0%|          | 0/1000 [00:00<?, ?it/s]Epoch #1: 9800it [00:30, 318.66it/s]              Epoch #1: 9800it [00:30, 318.63it/s]
Traceback (most recent call last):
  File "/nas/ucb/mason/ethically-compliant-rl/CPO/train_cpo.py", line 237, in <module>
    train()
  File "/nas/ucb/mason/ethically-compliant-rl/.venv/lib/python3.9/site-packages/pyrallis/argparsing.py", line 158, in wrapper_inner
    response = fn(cfg, *args, **kwargs)
  File "/nas/ucb/mason/ethically-compliant-rl/CPO/train_cpo.py", line 202, in train
    agent.learn(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/agent/cpo_agent.py", line 229, in learn
    return super().learn(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/agent/base_agent.py", line 353, in learn
    for epoch, _epoch_stat, info in trainer:
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/base_trainer.py", line 208, in __next__
    self.policy_update_fn(stats_train)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/onpolicy.py", line 104, in policy_update_fn
    self.policy.update(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 361, in update
    batch = self.process_fn(batch, buffer, indices)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/cpo.py", line 130, in process_fn
    batch = self.compute_gae_returns(batch, buffer, indices, self._lambda)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 421, in compute_gae_returns
    metrics = self.get_metrics(batch)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 392, in get_metrics
    metrics = [batch.rew, cost[:, 0], cost[:, 1]]
IndexError: index 1 is out of bounds for axis 1 with size 1
Epoch #1:   0%|          | 0/1000 [00:00<?, ?it/s]Epoch #1: 9051it [00:33, 270.07it/s]              Epoch #1: 9051it [00:33, 270.03it/s]
Traceback (most recent call last):
  File "/nas/ucb/mason/ethically-compliant-rl/CVPO/train_cvpo.py", line 246, in <module>
    train()
  File "/nas/ucb/mason/ethically-compliant-rl/.venv/lib/python3.9/site-packages/pyrallis/argparsing.py", line 158, in wrapper_inner
    response = fn(cfg, *args, **kwargs)
  File "/nas/ucb/mason/ethically-compliant-rl/CVPO/train_cvpo.py", line 211, in train
    agent.learn(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/agent/base_agent.py", line 236, in learn
    for epoch, _epoch_stat, info in trainer:
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/base_trainer.py", line 208, in __next__
    self.policy_update_fn(stats_train)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/offpolicy.py", line 106, in policy_update_fn
    self.policy.update(self.batch_size, self.train_collector.buffer)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 361, in update
    batch = self.process_fn(batch, buffer, indices)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/cvpo.py", line 216, in process_fn
    batch = self.compute_nstep_returns(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 494, in compute_nstep_returns
    metrics = self.get_metrics(buffer)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 392, in get_metrics
    metrics = [batch.rew, cost[:, 0], cost[:, 1]]
IndexError: index 1 is out of bounds for axis 1 with size 1
2023-08-23 22:58:14,656 - wandb.wandb_agent - INFO - Cleaning up finished run: h7abijfw
wandb: Waiting for W&B process to finish... (failed 1). Press Control-C to abort syncing.
Epoch #1:   0%|          | 0/1000 [00:00<?, ?it/s]Epoch #1: 9550it [00:36, 260.41it/s]              Epoch #1: 9550it [00:36, 260.39it/s]
Traceback (most recent call last):
  File "/nas/ucb/mason/ethically-compliant-rl/CVPO/train_cvpo.py", line 246, in <module>
    train()
  File "/nas/ucb/mason/ethically-compliant-rl/.venv/lib/python3.9/site-packages/pyrallis/argparsing.py", line 158, in wrapper_inner
    response = fn(cfg, *args, **kwargs)
  File "/nas/ucb/mason/ethically-compliant-rl/CVPO/train_cvpo.py", line 211, in train
    agent.learn(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/agent/base_agent.py", line 236, in learn
    for epoch, _epoch_stat, info in trainer:
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/base_trainer.py", line 208, in __next__
    self.policy_update_fn(stats_train)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/offpolicy.py", line 106, in policy_update_fn
    self.policy.update(self.batch_size, self.train_collector.buffer)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 361, in update
    batch = self.process_fn(batch, buffer, indices)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/cvpo.py", line 216, in process_fn
    batch = self.compute_nstep_returns(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 494, in compute_nstep_returns
    metrics = self.get_metrics(buffer)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 392, in get_metrics
    metrics = [batch.rew, cost[:, 0], cost[:, 1]]
IndexError: index 1 is out of bounds for axis 1 with size 1
2023-08-23 15:58:15,270 - wandb.wandb_agent - INFO - Cleaning up finished run: zdual6e6
wandb: Waiting for W&B process to finish... (failed 1). Press Control-C to abort syncing.
wandb: - 0.027 MB of 0.027 MB uploaded (0.000 MB deduped)wandb: \ 0.027 MB of 0.031 MB uploaded (0.000 MB deduped)wandb: üöÄ View run skilled-sweep-1 at: https://wandb.ai/ecrl/fast-safe-rl/runs/h7abijfw
wandb: Synced 5 W&B file(s), 0 media file(s), 0 artifact file(s) and 0 other file(s)
wandb: Find logs at: ./wandb/run-20230823_225731-h7abijfw/logs
Epoch #1:   0%|          | 0/1000 [00:00<?, ?it/s]Epoch #1: 10941it [00:38, 285.52it/s]             Epoch #1: 10941it [00:38, 285.49it/s]
Traceback (most recent call last):
  File "/nas/ucb/mason/ethically-compliant-rl/CPO/train_cpo.py", line 237, in <module>
    train()
  File "/nas/ucb/mason/ethically-compliant-rl/.venv/lib/python3.9/site-packages/pyrallis/argparsing.py", line 158, in wrapper_inner
    response = fn(cfg, *args, **kwargs)
  File "/nas/ucb/mason/ethically-compliant-rl/CPO/train_cpo.py", line 202, in train
    agent.learn(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/agent/cpo_agent.py", line 229, in learn
    return super().learn(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/agent/base_agent.py", line 353, in learn
    for epoch, _epoch_stat, info in trainer:
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/base_trainer.py", line 208, in __next__
    self.policy_update_fn(stats_train)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/onpolicy.py", line 104, in policy_update_fn
    self.policy.update(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 361, in update
    batch = self.process_fn(batch, buffer, indices)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/cpo.py", line 130, in process_fn
    batch = self.compute_gae_returns(batch, buffer, indices, self._lambda)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 421, in compute_gae_returns
    metrics = self.get_metrics(batch)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 392, in get_metrics
    metrics = [batch.rew, cost[:, 0], cost[:, 1]]
IndexError: index 1 is out of bounds for axis 1 with size 1
2023-08-23 22:58:19,655 - wandb.wandb_agent - INFO - Agent received command: run
2023-08-23 22:58:19,655 - wandb.wandb_agent - INFO - Agent starting run with config:
	gamma: 0.095
	lr: 0.001
2023-08-23 22:58:19,658 - wandb.wandb_agent - INFO - About to run command: /usr/bin/env python /nas/ucb/mason/ethically-compliant-rl/CPO/train_cpo.py --gamma=0.095 --lr=0.001
wandb: üöÄ View run hardy-sweep-3 at: https://wandb.ai/ecrl/fast-safe-rl/runs/zdual6e6
wandb: Synced 5 W&B file(s), 0 media file(s), 0 artifact file(s) and 0 other file(s)
wandb: Find logs at: ./wandb/run-20230823_155732-zdual6e6/logs
2023-08-23 22:58:20,088 - wandb.wandb_agent - INFO - Cleaning up finished run: q17z0y00
wandb: Waiting for W&B process to finish... (failed 1). Press Control-C to abort syncing.
2023-08-23 15:58:22,165 - wandb.wandb_agent - INFO - Agent received command: run
2023-08-23 15:58:22,165 - wandb.wandb_agent - INFO - Agent starting run with config:
	actor_lr: 0.0005
	critic_lr: 0.001
	gamma: 0.095
2023-08-23 15:58:22,166 - wandb.wandb_agent - INFO - About to run command: /usr/bin/env python /nas/ucb/mason/ethically-compliant-rl/CVPO/train_cvpo.py --actor_lr=0.0005 --critic_lr=0.001 --gamma=0.095
wandb: Currently logged in as: mason-nakamura1. Use `wandb login --relogin` to force relogin
wandb: WARNING Ignored wandb.init() arg project when running a sweep.
wandb: WARNING Ignored wandb.init() arg entity when running a sweep.
wandb: wandb version 0.15.8 is available!  To upgrade, please run:
wandb:  $ pip install wandb --upgrade
wandb: Tracking run with wandb version 0.14.2
wandb: Run data is saved locally in /nas/ucb/mason/ethically-compliant-rl/wandb/run-20230823_225822-j40yjv8v
wandb: Run `wandb offline` to turn off syncing.
wandb: Syncing run firm-sweep-4
wandb: ‚≠êÔ∏è View project at https://wandb.ai/ecrl/fast-safe-rl
wandb: üßπ View sweep at https://wandb.ai/ecrl/fast-safe-rl/sweeps/tea6k681
wandb: üöÄ View run at https://wandb.ai/ecrl/fast-safe-rl/runs/j40yjv8v
wandb: üöÄ View run hardy-sweep-2 at: https://wandb.ai/ecrl/fast-safe-rl/runs/q17z0y00
wandb: Synced 5 W&B file(s), 0 media file(s), 0 artifact file(s) and 0 other file(s)
wandb: Find logs at: ./wandb/run-20230823_225731-q17z0y00/logs
2023-08-23 22:58:24,665 - wandb.wandb_agent - INFO - Running runs: ['j40yjv8v']
wandb: Currently logged in as: mason-nakamura1. Use `wandb login --relogin` to force relogin
wandb: WARNING Ignored wandb.init() arg project when running a sweep.
wandb: WARNING Ignored wandb.init() arg entity when running a sweep.
2023-08-23 15:58:25,494 - wandb.wandb_agent - INFO - Cleaning up finished run: 0obizknr
wandb: Waiting for W&B process to finish... (failed 1). Press Control-C to abort syncing.
wandb: wandb version 0.15.8 is available!  To upgrade, please run:
wandb:  $ pip install wandb --upgrade
wandb: Tracking run with wandb version 0.14.2
wandb: Run data is saved locally in /nas/ucb/mason/ethically-compliant-rl/wandb/run-20230823_155825-gisu6ovi
wandb: Run `wandb offline` to turn off syncing.
wandb: Syncing run smooth-sweep-4
wandb: ‚≠êÔ∏è View project at https://wandb.ai/ecrl/fast-safe-rl
wandb: üßπ View sweep at https://wandb.ai/ecrl/fast-safe-rl/sweeps/3qe1o66j
wandb: üöÄ View run at https://wandb.ai/ecrl/fast-safe-rl/runs/gisu6ovi
2023-08-23 22:58:26,071 - wandb.wandb_agent - INFO - Agent received command: run
2023-08-23 22:58:26,072 - wandb.wandb_agent - INFO - Agent starting run with config:
	actor_lr: 0.001
	critic_lr: 0.001
	gamma: 0.099
2023-08-23 22:58:26,074 - wandb.wandb_agent - INFO - About to run command: /usr/bin/env python /nas/ucb/mason/ethically-compliant-rl/CVPO/train_cvpo.py --actor_lr=0.001 --critic_lr=0.001 --gamma=0.099
[32;1mLogging data to logs/CPO-sweep-700epochs/parking-v0-cost-[5.0]/cpo_constraint_typelines_cost5.0_gamma0.095_step_per_epoch1000-a663/progress.txt[0m
2023-08-23 15:58:27,175 - wandb.wandb_agent - INFO - Running runs: ['gisu6ovi']
wandb: - 0.027 MB of 0.027 MB uploaded (0.000 MB deduped)wandb: \ 0.027 MB of 0.031 MB uploaded (0.000 MB deduped)wandb: üöÄ View run azure-sweep-3 at: https://wandb.ai/ecrl/fast-safe-rl/runs/0obizknr
wandb: Synced 5 W&B file(s), 0 media file(s), 0 artifact file(s) and 0 other file(s)
wandb: Find logs at: ./wandb/run-20230823_155732-0obizknr/logs
wandb: Currently logged in as: mason-nakamura1. Use `wandb login --relogin` to force relogin
wandb: WARNING Ignored wandb.init() arg project when running a sweep.
wandb: WARNING Ignored wandb.init() arg entity when running a sweep.
wandb: wandb version 0.15.8 is available!  To upgrade, please run:
wandb:  $ pip install wandb --upgrade
wandb: Tracking run with wandb version 0.14.2
wandb: Run data is saved locally in /nas/ucb/mason/ethically-compliant-rl/wandb/run-20230823_225828-gohzvnwu
wandb: Run `wandb offline` to turn off syncing.
wandb: Syncing run gentle-sweep-5
wandb: ‚≠êÔ∏è View project at https://wandb.ai/ecrl/fast-safe-rl
wandb: üßπ View sweep at https://wandb.ai/ecrl/fast-safe-rl/sweeps/3qe1o66j
wandb: üöÄ View run at https://wandb.ai/ecrl/fast-safe-rl/runs/gohzvnwu
[32;1mLogging data to logs/CVPO-sweep-700epochs/parking-v0-cost-[5.0]/cvpo_constraint_typelines_cost5.0_gamma0.095_step_per_epoch1000-2a43/progress.txt[0m
2023-08-23 15:58:30,219 - wandb.wandb_agent - INFO - Agent received command: run
2023-08-23 15:58:30,220 - wandb.wandb_agent - INFO - Agent starting run with config:
	gamma: 0.095
	lr: 0.0005
2023-08-23 15:58:30,222 - wandb.wandb_agent - INFO - About to run command: /usr/bin/env python /nas/ucb/mason/ethically-compliant-rl/CPO/train_cpo.py --gamma=0.095 --lr=0.0005
2023-08-23 22:58:31,083 - wandb.wandb_agent - INFO - Running runs: ['gohzvnwu']
Epoch #1:   0%|          | 0/1000 [00:00<?, ?it/s]Epoch #1: 12043it [00:51, 235.50it/s]             Epoch #1: 12043it [00:51, 235.47it/s]
Traceback (most recent call last):
  File "/nas/ucb/mason/ethically-compliant-rl/CPO/train_cpo.py", line 237, in <module>
    train()
  File "/nas/ucb/mason/ethically-compliant-rl/.venv/lib/python3.9/site-packages/pyrallis/argparsing.py", line 158, in wrapper_inner
    response = fn(cfg, *args, **kwargs)
  File "/nas/ucb/mason/ethically-compliant-rl/CPO/train_cpo.py", line 202, in train
    agent.learn(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/agent/cpo_agent.py", line 229, in learn
    return super().learn(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/agent/base_agent.py", line 353, in learn
    for epoch, _epoch_stat, info in trainer:
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/base_trainer.py", line 208, in __next__
    self.policy_update_fn(stats_train)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/onpolicy.py", line 104, in policy_update_fn
    self.policy.update(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 361, in update
    batch = self.process_fn(batch, buffer, indices)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/cpo.py", line 130, in process_fn
    batch = self.compute_gae_returns(batch, buffer, indices, self._lambda)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 421, in compute_gae_returns
    metrics = self.get_metrics(batch)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 392, in get_metrics
    metrics = [batch.rew, cost[:, 0], cost[:, 1]]
IndexError: index 1 is out of bounds for axis 1 with size 1
[32;1mLogging data to logs/CVPO-sweep-700epochs/parking-v0-cost-[5.0]/cvpo_actor_lr0.001_constraint_typelines_cost5.0_gamma0.099_step_per_epoch1000-d029/progress.txt[0m
Epoch #1:   0%|          | 0/1000 [00:00<?, ?it/s]Epoch #1: 10612it [00:53, 198.13it/s]             Epoch #1: 10612it [00:53, 198.12it/s]
Traceback (most recent call last):
  File "/nas/ucb/mason/ethically-compliant-rl/CVPO/train_cvpo.py", line 246, in <module>
    train()
  File "/nas/ucb/mason/ethically-compliant-rl/.venv/lib/python3.9/site-packages/pyrallis/argparsing.py", line 158, in wrapper_inner
    response = fn(cfg, *args, **kwargs)
  File "/nas/ucb/mason/ethically-compliant-rl/CVPO/train_cvpo.py", line 211, in train
    agent.learn(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/agent/base_agent.py", line 236, in learn
    for epoch, _epoch_stat, info in trainer:
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/base_trainer.py", line 208, in __next__
    self.policy_update_fn(stats_train)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/offpolicy.py", line 106, in policy_update_fn
    self.policy.update(self.batch_size, self.train_collector.buffer)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 361, in update
    batch = self.process_fn(batch, buffer, indices)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/cvpo.py", line 216, in process_fn
    batch = self.compute_nstep_returns(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 494, in compute_nstep_returns
    metrics = self.get_metrics(buffer)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 392, in get_metrics
    metrics = [batch.rew, cost[:, 0], cost[:, 1]]
IndexError: index 1 is out of bounds for axis 1 with size 1
wandb: Currently logged in as: mason-nakamura1. Use `wandb login --relogin` to force relogin
wandb: WARNING Ignored wandb.init() arg project when running a sweep.
wandb: WARNING Ignored wandb.init() arg entity when running a sweep.
wandb: wandb version 0.15.8 is available!  To upgrade, please run:
wandb:  $ pip install wandb --upgrade
wandb: Tracking run with wandb version 0.14.2
wandb: Run data is saved locally in /nas/ucb/mason/ethically-compliant-rl/wandb/run-20230823_155833-8177kp9y
wandb: Run `wandb offline` to turn off syncing.
wandb: Syncing run fancy-sweep-5
wandb: ‚≠êÔ∏è View project at https://wandb.ai/ecrl/fast-safe-rl
wandb: üßπ View sweep at https://wandb.ai/ecrl/fast-safe-rl/sweeps/tea6k681
wandb: üöÄ View run at https://wandb.ai/ecrl/fast-safe-rl/runs/8177kp9y
2023-08-23 15:58:35,054 - wandb.wandb_agent - INFO - Cleaning up finished run: fktjssxy
wandb: Waiting for W&B process to finish... (failed 1). Press Control-C to abort syncing.
2023-08-23 15:58:35,231 - wandb.wandb_agent - INFO - Running runs: ['8177kp9y']
2023-08-23 15:58:35,391 - wandb.wandb_agent - INFO - Cleaning up finished run: rybmrwkh
wandb: Waiting for W&B process to finish... (failed 1). Press Control-C to abort syncing.
[32;1mLogging data to logs/CPO-sweep-700epochs/parking-v0-cost-[5.0]/cpo_constraint_typelines_cost5.0_gamma0.095_lr0.0005_step_per_epoch1000-66e8/progress.txt[0m
wandb: - 0.027 MB of 0.027 MB uploaded (0.000 MB deduped)wandb: \ 0.027 MB of 0.031 MB uploaded (0.000 MB deduped)wandb: üöÄ View run decent-sweep-2 at: https://wandb.ai/ecrl/fast-safe-rl/runs/rybmrwkh
wandb: Synced 5 W&B file(s), 0 media file(s), 0 artifact file(s) and 0 other file(s)
wandb: Find logs at: ./wandb/run-20230823_155731-rybmrwkh/logs
wandb: üöÄ View run celestial-sweep-1 at: https://wandb.ai/ecrl/fast-safe-rl/runs/fktjssxy
wandb: Synced 5 W&B file(s), 0 media file(s), 0 artifact file(s) and 0 other file(s)
wandb: Find logs at: ./wandb/run-20230823_155731-fktjssxy/logs
2023-08-23 15:58:41,110 - wandb.wandb_agent - INFO - Agent received command: run
2023-08-23 15:58:41,110 - wandb.wandb_agent - INFO - Agent starting run with config:
	gamma: 0.099
	lr: 0.0005
2023-08-23 15:58:41,113 - wandb.wandb_agent - INFO - About to run command: /usr/bin/env python /nas/ucb/mason/ethically-compliant-rl/CPO/train_cpo.py --gamma=0.099 --lr=0.0005
2023-08-23 15:58:41,319 - wandb.wandb_agent - INFO - Agent received command: run
2023-08-23 15:58:41,319 - wandb.wandb_agent - INFO - Agent starting run with config:
	actor_lr: 0.001
	critic_lr: 0.0005
	gamma: 0.095
2023-08-23 15:58:41,321 - wandb.wandb_agent - INFO - About to run command: /usr/bin/env python /nas/ucb/mason/ethically-compliant-rl/CVPO/train_cvpo.py --actor_lr=0.001 --critic_lr=0.0005 --gamma=0.095
wandb: Currently logged in as: mason-nakamura1. Use `wandb login --relogin` to force relogin
wandb: WARNING Ignored wandb.init() arg project when running a sweep.
wandb: WARNING Ignored wandb.init() arg entity when running a sweep.
wandb: wandb version 0.15.8 is available!  To upgrade, please run:
wandb:  $ pip install wandb --upgrade
wandb: Tracking run with wandb version 0.14.2
wandb: Run data is saved locally in /nas/ucb/mason/ethically-compliant-rl/wandb/run-20230823_155843-8ykk17e5
wandb: Run `wandb offline` to turn off syncing.
wandb: Syncing run glorious-sweep-6
wandb: ‚≠êÔ∏è View project at https://wandb.ai/ecrl/fast-safe-rl
wandb: üßπ View sweep at https://wandb.ai/ecrl/fast-safe-rl/sweeps/tea6k681
wandb: üöÄ View run at https://wandb.ai/ecrl/fast-safe-rl/runs/8ykk17e5
wandb: Currently logged in as: mason-nakamura1. Use `wandb login --relogin` to force relogin
wandb: WARNING Ignored wandb.init() arg project when running a sweep.
wandb: WARNING Ignored wandb.init() arg entity when running a sweep.
wandb: wandb version 0.15.8 is available!  To upgrade, please run:
wandb:  $ pip install wandb --upgrade
wandb: Tracking run with wandb version 0.14.2
wandb: Run data is saved locally in /nas/ucb/mason/ethically-compliant-rl/wandb/run-20230823_155844-ahjifcnq
wandb: Run `wandb offline` to turn off syncing.
wandb: Syncing run light-sweep-6
wandb: ‚≠êÔ∏è View project at https://wandb.ai/ecrl/fast-safe-rl
wandb: üßπ View sweep at https://wandb.ai/ecrl/fast-safe-rl/sweeps/3qe1o66j
wandb: üöÄ View run at https://wandb.ai/ecrl/fast-safe-rl/runs/ahjifcnq
2023-08-23 15:58:46,123 - wandb.wandb_agent - INFO - Running runs: ['8ykk17e5']
2023-08-23 15:58:46,328 - wandb.wandb_agent - INFO - Running runs: ['ahjifcnq']
[32;1mLogging data to logs/CPO-sweep-700epochs/parking-v0-cost-[5.0]/cpo_constraint_typelines_cost5.0_gamma0.099_lr0.0005_step_per_epoch1000-4828/progress.txt[0m
[32;1mLogging data to logs/CVPO-sweep-700epochs/parking-v0-cost-[5.0]/cvpo_actor_lr0.001_constraint_typelines_cost5.0_critic_lr0.0005_gamma0.095_step_per_epoch1000-34c0/progress.txt[0m
Epoch #1:   0%|          | 0/1000 [00:00<?, ?it/s]Epoch #1: 8190it [00:29, 273.84it/s]              Epoch #1: 8190it [00:29, 273.81it/s]
Traceback (most recent call last):
  File "/nas/ucb/mason/ethically-compliant-rl/CVPO/train_cvpo.py", line 246, in <module>
    train()
  File "/nas/ucb/mason/ethically-compliant-rl/.venv/lib/python3.9/site-packages/pyrallis/argparsing.py", line 158, in wrapper_inner
    response = fn(cfg, *args, **kwargs)
  File "/nas/ucb/mason/ethically-compliant-rl/CVPO/train_cvpo.py", line 211, in train
    agent.learn(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/agent/base_agent.py", line 236, in learn
    for epoch, _epoch_stat, info in trainer:
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/base_trainer.py", line 208, in __next__
    self.policy_update_fn(stats_train)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/offpolicy.py", line 106, in policy_update_fn
    self.policy.update(self.batch_size, self.train_collector.buffer)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 361, in update
    batch = self.process_fn(batch, buffer, indices)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/cvpo.py", line 216, in process_fn
    batch = self.compute_nstep_returns(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 494, in compute_nstep_returns
    metrics = self.get_metrics(buffer)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 392, in get_metrics
    metrics = [batch.rew, cost[:, 0], cost[:, 1]]
IndexError: index 1 is out of bounds for axis 1 with size 1
Traceback (most recent call last):
  File "/nas/ucb/mason/ethically-compliant-rl/CVPO/train_cvpo.py", line 246, in <module>
    train()
  File "/nas/ucb/mason/ethically-compliant-rl/.venv/lib/python3.9/site-packages/pyrallis/argparsing.py", line 158, in wrapper_inner
    response = fn(cfg, *args, **kwargs)
  File "/nas/ucb/mason/ethically-compliant-rl/CVPO/train_cvpo.py", line 211, in train
    agent.learn(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/agent/base_agent.py", line 236, in learn
    for epoch, _epoch_stat, info in trainer:
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/base_trainer.py", line 208, in __next__
    self.policy_update_fn(stats_train)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/offpolicy.py", line 106, in policy_update_fn
    self.policy.update(self.batch_size, self.train_collector.buffer)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 361, in update
    batch = self.process_fn(batch, buffer, indices)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/cvpo.py", line 216, in process_fn
    batch = self.compute_nstep_returns(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 494, in compute_nstep_returns
    metrics = self.get_metrics(buffer)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 392, in get_metrics
    metrics = [batch.rew, cost[:, 0], cost[:, 1]]
IndexError: index 1 is out of bounds for axis 1 with size 1
wandb: Waiting for W&B process to finish... (failed 1). Press Control-C to abort syncing.
Epoch #1:   0%|          | 0/1000 [00:00<?, ?it/s]Epoch #1: 11716it [00:35, 329.19it/s]             Epoch #1: 11716it [00:35, 329.15it/s]
Traceback (most recent call last):
  File "/nas/ucb/mason/ethically-compliant-rl/CPO/train_cpo.py", line 237, in <module>
    train()
  File "/nas/ucb/mason/ethically-compliant-rl/.venv/lib/python3.9/site-packages/pyrallis/argparsing.py", line 158, in wrapper_inner
    response = fn(cfg, *args, **kwargs)
  File "/nas/ucb/mason/ethically-compliant-rl/CPO/train_cpo.py", line 202, in train
    agent.learn(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/agent/cpo_agent.py", line 229, in learn
    return super().learn(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/agent/base_agent.py", line 353, in learn
    for epoch, _epoch_stat, info in trainer:
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/base_trainer.py", line 208, in __next__
    self.policy_update_fn(stats_train)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/onpolicy.py", line 104, in policy_update_fn
    self.policy.update(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 361, in update
    batch = self.process_fn(batch, buffer, indices)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/cpo.py", line 130, in process_fn
    batch = self.compute_gae_returns(batch, buffer, indices, self._lambda)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 421, in compute_gae_returns
    metrics = self.get_metrics(batch)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 392, in get_metrics
    metrics = [batch.rew, cost[:, 0], cost[:, 1]]
IndexError: index 1 is out of bounds for axis 1 with size 1
wandb: Waiting for W&B process to finish... (failed 1). Press Control-C to abort syncing.
wandb: üöÄ View run smooth-sweep-4 at: https://wandb.ai/ecrl/fast-safe-rl/runs/gisu6ovi
wandb: Synced 5 W&B file(s), 0 media file(s), 0 artifact file(s) and 0 other file(s)
wandb: Find logs at: ./wandb/run-20230823_155825-gisu6ovi/logs
wandb: üöÄ View run firm-sweep-4 at: https://wandb.ai/ecrl/fast-safe-rl/runs/j40yjv8v
wandb: Synced 5 W&B file(s), 0 media file(s), 0 artifact file(s) and 0 other file(s)
wandb: Find logs at: ./wandb/run-20230823_225822-j40yjv8v/logs
Epoch #1:   0%|          | 0/1000 [00:00<?, ?it/s]Epoch #1: 9991it [00:37, 269.26it/s]              Epoch #1: 9991it [00:37, 269.24it/s]
Traceback (most recent call last):
  File "/nas/ucb/mason/ethically-compliant-rl/CVPO/train_cvpo.py", line 246, in <module>
    train()
  File "/nas/ucb/mason/ethically-compliant-rl/.venv/lib/python3.9/site-packages/pyrallis/argparsing.py", line 158, in wrapper_inner
    response = fn(cfg, *args, **kwargs)
  File "/nas/ucb/mason/ethically-compliant-rl/CVPO/train_cvpo.py", line 211, in train
    agent.learn(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/agent/base_agent.py", line 236, in learn
    for epoch, _epoch_stat, info in trainer:
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/base_trainer.py", line 208, in __next__
    self.policy_update_fn(stats_train)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/offpolicy.py", line 106, in policy_update_fn
    self.policy.update(self.batch_size, self.train_collector.buffer)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 361, in update
    batch = self.process_fn(batch, buffer, indices)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/cvpo.py", line 216, in process_fn
    batch = self.compute_nstep_returns(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 494, in compute_nstep_returns
    metrics = self.get_metrics(buffer)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 392, in get_metrics
    metrics = [batch.rew, cost[:, 0], cost[:, 1]]
IndexError: index 1 is out of bounds for axis 1 with size 1
Traceback (most recent call last):
  File "/nas/ucb/mason/ethically-compliant-rl/CVPO/train_cvpo.py", line 246, in <module>
    train()
  File "/nas/ucb/mason/ethically-compliant-rl/.venv/lib/python3.9/site-packages/pyrallis/argparsing.py", line 158, in wrapper_inner
    response = fn(cfg, *args, **kwargs)
  File "/nas/ucb/mason/ethically-compliant-rl/CVPO/train_cvpo.py", line 211, in train
    agent.learn(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/agent/base_agent.py", line 236, in learn
    for epoch, _epoch_stat, info in trainer:
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/base_trainer.py", line 208, in __next__
    self.policy_update_fn(stats_train)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/offpolicy.py", line 106, in policy_update_fn
    self.policy.update(self.batch_size, self.train_collector.buffer)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 361, in update
    batch = self.process_fn(batch, buffer, indices)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/cvpo.py", line 216, in process_fn
    batch = self.compute_nstep_returns(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 494, in compute_nstep_returns
    metrics = self.get_metrics(buffer)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 392, in get_metrics
    metrics = [batch.rew, cost[:, 0], cost[:, 1]]
IndexError: index 1 is out of bounds for axis 1 with size 1
wandb: Waiting for W&B process to finish... (failed 1). Press Control-C to abort syncing.
2023-08-23 15:59:12,967 - wandb.wandb_agent - INFO - Cleaning up finished run: gisu6ovi
2023-08-23 15:59:13,401 - wandb.wandb_agent - INFO - Agent received command: run
2023-08-23 15:59:13,401 - wandb.wandb_agent - INFO - Agent starting run with config:
	actor_lr: 0.0005
	critic_lr: 0.001
	gamma: 0.099
2023-08-23 15:59:13,404 - wandb.wandb_agent - INFO - About to run command: /usr/bin/env python /nas/ucb/mason/ethically-compliant-rl/CVPO/train_cvpo.py --actor_lr=0.0005 --critic_lr=0.001 --gamma=0.099
wandb: - 0.027 MB of 0.033 MB uploaded (0.000 MB deduped)wandb: üöÄ View run gentle-sweep-5 at: https://wandb.ai/ecrl/fast-safe-rl/runs/gohzvnwu
wandb: Synced 5 W&B file(s), 0 media file(s), 0 artifact file(s) and 0 other file(s)
wandb: Find logs at: ./wandb/run-20230823_225828-gohzvnwu/logs
Epoch #1:   0%|          | 0/1000 [00:00<?, ?it/s]Epoch #1: 9548it [00:32, 289.60it/s]              Epoch #1: 9548it [00:32, 289.57it/s]
Traceback (most recent call last):
  File "/nas/ucb/mason/ethically-compliant-rl/CPO/train_cpo.py", line 237, in <module>
    train()
  File "/nas/ucb/mason/ethically-compliant-rl/.venv/lib/python3.9/site-packages/pyrallis/argparsing.py", line 158, in wrapper_inner
    response = fn(cfg, *args, **kwargs)
  File "/nas/ucb/mason/ethically-compliant-rl/CPO/train_cpo.py", line 202, in train
    agent.learn(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/agent/cpo_agent.py", line 229, in learn
    return super().learn(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/agent/base_agent.py", line 353, in learn
    for epoch, _epoch_stat, info in trainer:
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/base_trainer.py", line 208, in __next__
    self.policy_update_fn(stats_train)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/onpolicy.py", line 104, in policy_update_fn
    self.policy.update(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 361, in update
    batch = self.process_fn(batch, buffer, indices)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/cpo.py", line 130, in process_fn
    batch = self.compute_gae_returns(batch, buffer, indices, self._lambda)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 421, in compute_gae_returns
    metrics = self.get_metrics(batch)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 392, in get_metrics
    metrics = [batch.rew, cost[:, 0], cost[:, 1]]
IndexError: index 1 is out of bounds for axis 1 with size 1
wandb: Waiting for W&B process to finish... (failed 1). Press Control-C to abort syncing.
2023-08-23 22:59:15,477 - wandb.wandb_agent - INFO - Cleaning up finished run: j40yjv8v
2023-08-23 22:59:15,941 - wandb.wandb_agent - INFO - Agent received command: run
2023-08-23 22:59:15,942 - wandb.wandb_agent - INFO - Agent starting run with config:
	gamma: 0.095
	lr: 0.0005
2023-08-23 22:59:15,943 - wandb.wandb_agent - INFO - About to run command: /usr/bin/env python /nas/ucb/mason/ethically-compliant-rl/CPO/train_cpo.py --gamma=0.095 --lr=0.0005
wandb: Currently logged in as: mason-nakamura1. Use `wandb login --relogin` to force relogin
wandb: WARNING Ignored wandb.init() arg project when running a sweep.
wandb: WARNING Ignored wandb.init() arg entity when running a sweep.
2023-08-23 22:59:16,899 - wandb.wandb_agent - INFO - Cleaning up finished run: gohzvnwu
wandb: wandb version 0.15.8 is available!  To upgrade, please run:
wandb:  $ pip install wandb --upgrade
wandb: Tracking run with wandb version 0.14.2
wandb: Run data is saved locally in /nas/ucb/mason/ethically-compliant-rl/wandb/run-20230823_155916-0qweed18
wandb: Run `wandb offline` to turn off syncing.
wandb: Syncing run legendary-sweep-7
wandb: ‚≠êÔ∏è View project at https://wandb.ai/ecrl/fast-safe-rl
wandb: üßπ View sweep at https://wandb.ai/ecrl/fast-safe-rl/sweeps/3qe1o66j
wandb: üöÄ View run at https://wandb.ai/ecrl/fast-safe-rl/runs/0qweed18
2023-08-23 22:59:17,428 - wandb.wandb_agent - INFO - Agent received command: run
2023-08-23 22:59:17,428 - wandb.wandb_agent - INFO - Agent starting run with config:
	actor_lr: 0.0005
	critic_lr: 0.0005
	gamma: 0.099
2023-08-23 22:59:17,430 - wandb.wandb_agent - INFO - About to run command: /usr/bin/env python /nas/ucb/mason/ethically-compliant-rl/CVPO/train_cvpo.py --actor_lr=0.0005 --critic_lr=0.0005 --gamma=0.099
wandb: - 0.027 MB of 0.027 MB uploaded (0.000 MB deduped)wandb: \ 0.031 MB of 0.031 MB uploaded (0.000 MB deduped)wandb: üöÄ View run fancy-sweep-5 at: https://wandb.ai/ecrl/fast-safe-rl/runs/8177kp9y
wandb: Synced 5 W&B file(s), 0 media file(s), 0 artifact file(s) and 0 other file(s)
wandb: Find logs at: ./wandb/run-20230823_155833-8177kp9y/logs
2023-08-23 15:59:18,414 - wandb.wandb_agent - INFO - Running runs: ['0qweed18']
wandb: Currently logged in as: mason-nakamura1. Use `wandb login --relogin` to force relogin
wandb: WARNING Ignored wandb.init() arg project when running a sweep.
wandb: WARNING Ignored wandb.init() arg entity when running a sweep.
wandb: wandb version 0.15.8 is available!  To upgrade, please run:
wandb:  $ pip install wandb --upgrade
wandb: Tracking run with wandb version 0.14.2
wandb: Run data is saved locally in /nas/ucb/mason/ethically-compliant-rl/wandb/run-20230823_225918-yxwhxxa7
wandb: Run `wandb offline` to turn off syncing.
wandb: Syncing run ancient-sweep-7
wandb: ‚≠êÔ∏è View project at https://wandb.ai/ecrl/fast-safe-rl
wandb: üßπ View sweep at https://wandb.ai/ecrl/fast-safe-rl/sweeps/tea6k681
wandb: üöÄ View run at https://wandb.ai/ecrl/fast-safe-rl/runs/yxwhxxa7
wandb: Currently logged in as: mason-nakamura1. Use `wandb login --relogin` to force relogin
wandb: WARNING Ignored wandb.init() arg project when running a sweep.
wandb: WARNING Ignored wandb.init() arg entity when running a sweep.
[32;1mLogging data to logs/CVPO-sweep-700epochs/parking-v0-cost-[5.0]/cvpo_constraint_typelines_cost5.0_gamma0.099_step_per_epoch1000-464d/progress.txt[0m
2023-08-23 22:59:20,952 - wandb.wandb_agent - INFO - Running runs: ['yxwhxxa7']
2023-08-23 15:59:21,022 - wandb.wandb_agent - INFO - Cleaning up finished run: 8177kp9y
wandb: wandb version 0.15.8 is available!  To upgrade, please run:
wandb:  $ pip install wandb --upgrade
wandb: Tracking run with wandb version 0.14.2
wandb: Run data is saved locally in /nas/ucb/mason/ethically-compliant-rl/wandb/run-20230823_225920-h3wa9i7n
wandb: Run `wandb offline` to turn off syncing.
wandb: Syncing run rural-sweep-8
wandb: ‚≠êÔ∏è View project at https://wandb.ai/ecrl/fast-safe-rl
wandb: üßπ View sweep at https://wandb.ai/ecrl/fast-safe-rl/sweeps/3qe1o66j
wandb: üöÄ View run at https://wandb.ai/ecrl/fast-safe-rl/runs/h3wa9i7n
2023-08-23 15:59:21,500 - wandb.wandb_agent - INFO - Agent received command: run
2023-08-23 15:59:21,500 - wandb.wandb_agent - INFO - Agent starting run with config:
	gamma: 0.095
	lr: 0.001
2023-08-23 15:59:21,502 - wandb.wandb_agent - INFO - About to run command: /usr/bin/env python /nas/ucb/mason/ethically-compliant-rl/CPO/train_cpo.py --gamma=0.095 --lr=0.001
2023-08-23 22:59:22,437 - wandb.wandb_agent - INFO - Running runs: ['h3wa9i7n']
[32;1mLogging data to logs/CPO-sweep-700epochs/parking-v0-cost-[5.0]/cpo_constraint_typelines_cost5.0_gamma0.095_lr0.0005_step_per_epoch1000-60d3/progress.txt[0m
[32;1mLogging data to logs/CVPO-sweep-700epochs/parking-v0-cost-[5.0]/cvpo_constraint_typelines_cost5.0_critic_lr0.0005_gamma0.099_step_per_epoch1000-3639/progress.txt[0m
wandb: Currently logged in as: mason-nakamura1. Use `wandb login --relogin` to force relogin
wandb: WARNING Ignored wandb.init() arg project when running a sweep.
wandb: WARNING Ignored wandb.init() arg entity when running a sweep.
wandb: wandb version 0.15.8 is available!  To upgrade, please run:
wandb:  $ pip install wandb --upgrade
wandb: Tracking run with wandb version 0.14.2
wandb: Run data is saved locally in /nas/ucb/mason/ethically-compliant-rl/wandb/run-20230823_155924-0nu523d4
wandb: Run `wandb offline` to turn off syncing.
wandb: Syncing run azure-sweep-8
wandb: ‚≠êÔ∏è View project at https://wandb.ai/ecrl/fast-safe-rl
wandb: üßπ View sweep at https://wandb.ai/ecrl/fast-safe-rl/sweeps/tea6k681
wandb: üöÄ View run at https://wandb.ai/ecrl/fast-safe-rl/runs/0nu523d4
2023-08-23 15:59:26,512 - wandb.wandb_agent - INFO - Running runs: ['0nu523d4']
Epoch #1:   0%|          | 0/1000 [00:00<?, ?it/s]Epoch #1: 10637it [00:34, 306.70it/s]             Epoch #1: 10637it [00:34, 306.66it/s]
Traceback (most recent call last):
  File "/nas/ucb/mason/ethically-compliant-rl/CPO/train_cpo.py", line 237, in <module>
    train()
  File "/nas/ucb/mason/ethically-compliant-rl/.venv/lib/python3.9/site-packages/pyrallis/argparsing.py", line 158, in wrapper_inner
    response = fn(cfg, *args, **kwargs)
  File "/nas/ucb/mason/ethically-compliant-rl/CPO/train_cpo.py", line 202, in train
    agent.learn(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/agent/cpo_agent.py", line 229, in learn
    return super().learn(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/agent/base_agent.py", line 353, in learn
    for epoch, _epoch_stat, info in trainer:
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/base_trainer.py", line 208, in __next__
    self.policy_update_fn(stats_train)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/onpolicy.py", line 104, in policy_update_fn
    self.policy.update(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 361, in update
    batch = self.process_fn(batch, buffer, indices)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/cpo.py", line 130, in process_fn
    batch = self.compute_gae_returns(batch, buffer, indices, self._lambda)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 421, in compute_gae_returns
    metrics = self.get_metrics(batch)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 392, in get_metrics
    metrics = [batch.rew, cost[:, 0], cost[:, 1]]
IndexError: index 1 is out of bounds for axis 1 with size 1
wandb: Waiting for W&B process to finish... (failed 1). Press Control-C to abort syncing.
[32;1mLogging data to logs/CPO-sweep-700epochs/parking-v0-cost-[5.0]/cpo_constraint_typelines_cost5.0_gamma0.095_step_per_epoch1000-1720/progress.txt[0m
wandb: - 0.027 MB of 0.027 MB uploaded (0.000 MB deduped)wandb: \ 0.027 MB of 0.030 MB uploaded (0.000 MB deduped)wandb: üöÄ View run glorious-sweep-6 at: https://wandb.ai/ecrl/fast-safe-rl/runs/8ykk17e5
wandb: Synced 5 W&B file(s), 0 media file(s), 0 artifact file(s) and 0 other file(s)
wandb: Find logs at: ./wandb/run-20230823_155843-8ykk17e5/logs
2023-08-23 15:59:37,027 - wandb.wandb_agent - INFO - Cleaning up finished run: 8ykk17e5
2023-08-23 15:59:37,513 - wandb.wandb_agent - INFO - Agent received command: run
2023-08-23 15:59:37,513 - wandb.wandb_agent - INFO - Agent starting run with config:
	gamma: 0.099
	lr: 0.001
2023-08-23 15:59:37,517 - wandb.wandb_agent - INFO - About to run command: /usr/bin/env python /nas/ucb/mason/ethically-compliant-rl/CPO/train_cpo.py --gamma=0.099 --lr=0.001
wandb: Currently logged in as: mason-nakamura1. Use `wandb login --relogin` to force relogin
wandb: WARNING Ignored wandb.init() arg project when running a sweep.
wandb: WARNING Ignored wandb.init() arg entity when running a sweep.
wandb: - Waiting for wandb.init()...wandb: \ Waiting for wandb.init()...wandb: wandb version 0.15.8 is available!  To upgrade, please run:
wandb:  $ pip install wandb --upgrade
wandb: Tracking run with wandb version 0.14.2
wandb: Run data is saved locally in /nas/ucb/mason/ethically-compliant-rl/wandb/run-20230823_155940-iav5e5r3
wandb: Run `wandb offline` to turn off syncing.
wandb: Syncing run rich-sweep-9
wandb: ‚≠êÔ∏è View project at https://wandb.ai/ecrl/fast-safe-rl
wandb: üßπ View sweep at https://wandb.ai/ecrl/fast-safe-rl/sweeps/tea6k681
wandb: üöÄ View run at https://wandb.ai/ecrl/fast-safe-rl/runs/iav5e5r3
2023-08-23 15:59:42,526 - wandb.wandb_agent - INFO - Running runs: ['iav5e5r3']
[32;1mLogging data to logs/CPO-sweep-700epochs/parking-v0-cost-[5.0]/cpo_constraint_typelines_cost5.0_gamma0.099_step_per_epoch1000-1f2b/progress.txt[0m
Epoch #1:   0%|          | 0/1000 [00:00<?, ?it/s]Epoch #1: 10737it [00:54, 196.53it/s]             Epoch #1: 10737it [00:54, 196.52it/s]
Traceback (most recent call last):
  File "/nas/ucb/mason/ethically-compliant-rl/CVPO/train_cvpo.py", line 246, in <module>
    train()
  File "/nas/ucb/mason/ethically-compliant-rl/.venv/lib/python3.9/site-packages/pyrallis/argparsing.py", line 158, in wrapper_inner
    response = fn(cfg, *args, **kwargs)
  File "/nas/ucb/mason/ethically-compliant-rl/CVPO/train_cvpo.py", line 211, in train
    agent.learn(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/agent/base_agent.py", line 236, in learn
    for epoch, _epoch_stat, info in trainer:
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/base_trainer.py", line 208, in __next__
    self.policy_update_fn(stats_train)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/offpolicy.py", line 106, in policy_update_fn
    self.policy.update(self.batch_size, self.train_collector.buffer)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 361, in update
    batch = self.process_fn(batch, buffer, indices)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/cvpo.py", line 216, in process_fn
    batch = self.compute_nstep_returns(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 494, in compute_nstep_returns
    metrics = self.get_metrics(buffer)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 392, in get_metrics
    metrics = [batch.rew, cost[:, 0], cost[:, 1]]
IndexError: index 1 is out of bounds for axis 1 with size 1
Traceback (most recent call last):
  File "/nas/ucb/mason/ethically-compliant-rl/CVPO/train_cvpo.py", line 246, in <module>
    train()
  File "/nas/ucb/mason/ethically-compliant-rl/.venv/lib/python3.9/site-packages/pyrallis/argparsing.py", line 158, in wrapper_inner
    response = fn(cfg, *args, **kwargs)
  File "/nas/ucb/mason/ethically-compliant-rl/CVPO/train_cvpo.py", line 211, in train
    agent.learn(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/agent/base_agent.py", line 236, in learn
    for epoch, _epoch_stat, info in trainer:
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/base_trainer.py", line 208, in __next__
    self.policy_update_fn(stats_train)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/offpolicy.py", line 106, in policy_update_fn
    self.policy.update(self.batch_size, self.train_collector.buffer)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 361, in update
    batch = self.process_fn(batch, buffer, indices)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/cvpo.py", line 216, in process_fn
    batch = self.compute_nstep_returns(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 494, in compute_nstep_returns
    metrics = self.get_metrics(buffer)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 392, in get_metrics
    metrics = [batch.rew, cost[:, 0], cost[:, 1]]
IndexError: index 1 is out of bounds for axis 1 with size 1
wandb: Waiting for W&B process to finish... (failed 1). Press Control-C to abort syncing.
wandb: üöÄ View run light-sweep-6 at: https://wandb.ai/ecrl/fast-safe-rl/runs/ahjifcnq
wandb: Synced 5 W&B file(s), 0 media file(s), 0 artifact file(s) and 0 other file(s)
wandb: Find logs at: ./wandb/run-20230823_155844-ahjifcnq/logs
2023-08-23 15:59:57,507 - wandb.wandb_agent - INFO - Cleaning up finished run: ahjifcnq
2023-08-23 15:59:57,952 - wandb.wandb_agent - INFO - Agent received command: run
2023-08-23 15:59:57,952 - wandb.wandb_agent - INFO - Agent starting run with config:
	actor_lr: 0.001
	critic_lr: 0.0005
	gamma: 0.099
2023-08-23 15:59:57,954 - wandb.wandb_agent - INFO - About to run command: /usr/bin/env python /nas/ucb/mason/ethically-compliant-rl/CVPO/train_cvpo.py --actor_lr=0.001 --critic_lr=0.0005 --gamma=0.099
Epoch #1:   0%|          | 0/1000 [00:00<?, ?it/s]Epoch #1: 8793it [00:36, 243.39it/s]              Epoch #1: 8793it [00:36, 243.37it/s]
Traceback (most recent call last):
  File "/nas/ucb/mason/ethically-compliant-rl/CVPO/train_cvpo.py", line 246, in <module>
    train()
  File "/nas/ucb/mason/ethically-compliant-rl/.venv/lib/python3.9/site-packages/pyrallis/argparsing.py", line 158, in wrapper_inner
    response = fn(cfg, *args, **kwargs)
  File "/nas/ucb/mason/ethically-compliant-rl/CVPO/train_cvpo.py", line 211, in train
    agent.learn(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/agent/base_agent.py", line 236, in learn
    for epoch, _epoch_stat, info in trainer:
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/base_trainer.py", line 208, in __next__
    self.policy_update_fn(stats_train)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/offpolicy.py", line 106, in policy_update_fn
    self.policy.update(self.batch_size, self.train_collector.buffer)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 361, in update
    batch = self.process_fn(batch, buffer, indices)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/cvpo.py", line 216, in process_fn
    batch = self.compute_nstep_returns(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 494, in compute_nstep_returns
    metrics = self.get_metrics(buffer)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 392, in get_metrics
    metrics = [batch.rew, cost[:, 0], cost[:, 1]]
IndexError: index 1 is out of bounds for axis 1 with size 1
Traceback (most recent call last):
  File "/nas/ucb/mason/ethically-compliant-rl/CVPO/train_cvpo.py", line 246, in <module>
    train()
  File "/nas/ucb/mason/ethically-compliant-rl/.venv/lib/python3.9/site-packages/pyrallis/argparsing.py", line 158, in wrapper_inner
    response = fn(cfg, *args, **kwargs)
  File "/nas/ucb/mason/ethically-compliant-rl/CVPO/train_cvpo.py", line 211, in train
    agent.learn(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/agent/base_agent.py", line 236, in learn
    for epoch, _epoch_stat, info in trainer:
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/base_trainer.py", line 208, in __next__
    self.policy_update_fn(stats_train)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/offpolicy.py", line 106, in policy_update_fn
    self.policy.update(self.batch_size, self.train_collector.buffer)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 361, in update
    batch = self.process_fn(batch, buffer, indices)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/cvpo.py", line 216, in process_fn
    batch = self.compute_nstep_returns(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 494, in compute_nstep_returns
    metrics = self.get_metrics(buffer)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 392, in get_metrics
    metrics = [batch.rew, cost[:, 0], cost[:, 1]]
IndexError: index 1 is out of bounds for axis 1 with size 1
wandb: Waiting for W&B process to finish... (failed 1). Press Control-C to abort syncing.
wandb: Currently logged in as: mason-nakamura1. Use `wandb login --relogin` to force relogin
wandb: WARNING Ignored wandb.init() arg project when running a sweep.
wandb: WARNING Ignored wandb.init() arg entity when running a sweep.
wandb: wandb version 0.15.8 is available!  To upgrade, please run:
wandb:  $ pip install wandb --upgrade
wandb: Tracking run with wandb version 0.14.2
wandb: Run data is saved locally in /nas/ucb/mason/ethically-compliant-rl/wandb/run-20230823_160001-qjjv7wrw
wandb: Run `wandb offline` to turn off syncing.
wandb: Syncing run celestial-sweep-9
wandb: ‚≠êÔ∏è View project at https://wandb.ai/ecrl/fast-safe-rl
wandb: üßπ View sweep at https://wandb.ai/ecrl/fast-safe-rl/sweeps/3qe1o66j
wandb: üöÄ View run at https://wandb.ai/ecrl/fast-safe-rl/runs/qjjv7wrw
2023-08-23 16:00:02,964 - wandb.wandb_agent - INFO - Running runs: ['qjjv7wrw']
wandb: - 0.027 MB of 0.027 MB uploaded (0.000 MB deduped)wandb: \ 0.027 MB of 0.030 MB uploaded (0.000 MB deduped)wandb: | 0.033 MB of 0.033 MB uploaded (0.000 MB deduped)wandb: üöÄ View run legendary-sweep-7 at: https://wandb.ai/ecrl/fast-safe-rl/runs/0qweed18
wandb: Synced 5 W&B file(s), 0 media file(s), 0 artifact file(s) and 0 other file(s)
wandb: Find logs at: ./wandb/run-20230823_155916-0qweed18/logs
Epoch #1:   0%|          | 0/1000 [00:00<?, ?it/s]Epoch #1: 11147it [00:36, 302.43it/s]             Epoch #1: 11147it [00:36, 302.40it/s]
Traceback (most recent call last):
  File "/nas/ucb/mason/ethically-compliant-rl/CPO/train_cpo.py", line 237, in <module>
    train()
  File "/nas/ucb/mason/ethically-compliant-rl/.venv/lib/python3.9/site-packages/pyrallis/argparsing.py", line 158, in wrapper_inner
    response = fn(cfg, *args, **kwargs)
  File "/nas/ucb/mason/ethically-compliant-rl/CPO/train_cpo.py", line 202, in train
    agent.learn(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/agent/cpo_agent.py", line 229, in learn
    return super().learn(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/agent/base_agent.py", line 353, in learn
    for epoch, _epoch_stat, info in trainer:
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/base_trainer.py", line 208, in __next__
    self.policy_update_fn(stats_train)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/onpolicy.py", line 104, in policy_update_fn
    self.policy.update(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 361, in update
    batch = self.process_fn(batch, buffer, indices)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/cpo.py", line 130, in process_fn
    batch = self.compute_gae_returns(batch, buffer, indices, self._lambda)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 421, in compute_gae_returns
    metrics = self.get_metrics(batch)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 392, in get_metrics
    metrics = [batch.rew, cost[:, 0], cost[:, 1]]
IndexError: index 1 is out of bounds for axis 1 with size 1
wandb: Waiting for W&B process to finish... (failed 1). Press Control-C to abort syncing.
[32;1mLogging data to logs/CVPO-sweep-700epochs/parking-v0-cost-[5.0]/cvpo_actor_lr0.001_constraint_typelines_cost5.0_critic_lr0.0005_gamma0.099_step_per_epoch1000-a6bf/progress.txt[0m
wandb: - 0.027 MB of 0.027 MB uploaded (0.000 MB deduped)wandb: \ 0.031 MB of 0.031 MB uploaded (0.000 MB deduped)wandb: üöÄ View run ancient-sweep-7 at: https://wandb.ai/ecrl/fast-safe-rl/runs/yxwhxxa7
wandb: Synced 5 W&B file(s), 0 media file(s), 0 artifact file(s) and 0 other file(s)
wandb: Find logs at: ./wandb/run-20230823_225918-yxwhxxa7/logs
Epoch #1:   0%|          | 0/1000 [00:00<?, ?it/s]Epoch #1: 9902it [00:34, 289.54it/s]              Epoch #1: 9902it [00:34, 289.51it/s]
Traceback (most recent call last):
  File "/nas/ucb/mason/ethically-compliant-rl/CPO/train_cpo.py", line 237, in <module>
    train()
  File "/nas/ucb/mason/ethically-compliant-rl/.venv/lib/python3.9/site-packages/pyrallis/argparsing.py", line 158, in wrapper_inner
    response = fn(cfg, *args, **kwargs)
  File "/nas/ucb/mason/ethically-compliant-rl/CPO/train_cpo.py", line 202, in train
    agent.learn(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/agent/cpo_agent.py", line 229, in learn
    return super().learn(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/agent/base_agent.py", line 353, in learn
    for epoch, _epoch_stat, info in trainer:
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/base_trainer.py", line 208, in __next__
    self.policy_update_fn(stats_train)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/onpolicy.py", line 104, in policy_update_fn
    self.policy.update(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 361, in update
    batch = self.process_fn(batch, buffer, indices)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/cpo.py", line 130, in process_fn
    batch = self.compute_gae_returns(batch, buffer, indices, self._lambda)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 421, in compute_gae_returns
    metrics = self.get_metrics(batch)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 392, in get_metrics
    metrics = [batch.rew, cost[:, 0], cost[:, 1]]
IndexError: index 1 is out of bounds for axis 1 with size 1
wandb: Waiting for W&B process to finish... (failed 1). Press Control-C to abort syncing.
Epoch #1:   0%|          | 0/1000 [00:00<?, ?it/s]Epoch #1: 11555it [00:40, 283.40it/s]             Epoch #1: 11555it [00:40, 283.38it/s]
Traceback (most recent call last):
  File "/nas/ucb/mason/ethically-compliant-rl/CVPO/train_cvpo.py", line 246, in <module>
    train()
  File "/nas/ucb/mason/ethically-compliant-rl/.venv/lib/python3.9/site-packages/pyrallis/argparsing.py", line 158, in wrapper_inner
    response = fn(cfg, *args, **kwargs)
  File "/nas/ucb/mason/ethically-compliant-rl/CVPO/train_cvpo.py", line 211, in train
    agent.learn(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/agent/base_agent.py", line 236, in learn
    for epoch, _epoch_stat, info in trainer:
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/base_trainer.py", line 208, in __next__
    self.policy_update_fn(stats_train)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/offpolicy.py", line 106, in policy_update_fn
    self.policy.update(self.batch_size, self.train_collector.buffer)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 361, in update
    batch = self.process_fn(batch, buffer, indices)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/cvpo.py", line 216, in process_fn
    batch = self.compute_nstep_returns(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 494, in compute_nstep_returns
    metrics = self.get_metrics(buffer)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 392, in get_metrics
    metrics = [batch.rew, cost[:, 0], cost[:, 1]]
IndexError: index 1 is out of bounds for axis 1 with size 1
Traceback (most recent call last):
  File "/nas/ucb/mason/ethically-compliant-rl/CVPO/train_cvpo.py", line 246, in <module>
    train()
  File "/nas/ucb/mason/ethically-compliant-rl/.venv/lib/python3.9/site-packages/pyrallis/argparsing.py", line 158, in wrapper_inner
    response = fn(cfg, *args, **kwargs)
  File "/nas/ucb/mason/ethically-compliant-rl/CVPO/train_cvpo.py", line 211, in train
    agent.learn(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/agent/base_agent.py", line 236, in learn
    for epoch, _epoch_stat, info in trainer:
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/base_trainer.py", line 208, in __next__
    self.policy_update_fn(stats_train)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/offpolicy.py", line 106, in policy_update_fn
    self.policy.update(self.batch_size, self.train_collector.buffer)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 361, in update
    batch = self.process_fn(batch, buffer, indices)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/cvpo.py", line 216, in process_fn
    batch = self.compute_nstep_returns(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 494, in compute_nstep_returns
    metrics = self.get_metrics(buffer)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 392, in get_metrics
    metrics = [batch.rew, cost[:, 0], cost[:, 1]]
IndexError: index 1 is out of bounds for axis 1 with size 1
wandb: Waiting for W&B process to finish... (failed 1). Press Control-C to abort syncing.
2023-08-23 16:00:09,323 - wandb.wandb_agent - INFO - Cleaning up finished run: 0qweed18
2023-08-23 16:00:09,784 - wandb.wandb_agent - INFO - Agent received command: run
2023-08-23 16:00:09,784 - wandb.wandb_agent - INFO - Agent starting run with config:
	actor_lr: 0.0005
	critic_lr: 0.0005
	gamma: 0.095
2023-08-23 16:00:09,787 - wandb.wandb_agent - INFO - About to run command: /usr/bin/env python /nas/ucb/mason/ethically-compliant-rl/CVPO/train_cvpo.py --actor_lr=0.0005 --critic_lr=0.0005 --gamma=0.095
wandb: - 0.027 MB of 0.027 MB uploaded (0.000 MB deduped)wandb: \ 0.027 MB of 0.029 MB uploaded (0.000 MB deduped)wandb: | 0.031 MB of 0.031 MB uploaded (0.000 MB deduped)wandb: üöÄ View run azure-sweep-8 at: https://wandb.ai/ecrl/fast-safe-rl/runs/0nu523d4
wandb: Synced 5 W&B file(s), 0 media file(s), 0 artifact file(s) and 0 other file(s)
wandb: Find logs at: ./wandb/run-20230823_155924-0nu523d4/logs
wandb: - 0.027 MB of 0.027 MB uploaded (0.000 MB deduped)wandb: \ 0.027 MB of 0.032 MB uploaded (0.000 MB deduped)wandb: | 0.033 MB of 0.033 MB uploaded (0.000 MB deduped)wandb: üöÄ View run rural-sweep-8 at: https://wandb.ai/ecrl/fast-safe-rl/runs/h3wa9i7n
wandb: Synced 5 W&B file(s), 0 media file(s), 0 artifact file(s) and 0 other file(s)
wandb: Find logs at: ./wandb/run-20230823_225920-h3wa9i7n/logs
2023-08-23 23:00:11,768 - wandb.wandb_agent - INFO - Cleaning up finished run: yxwhxxa7
2023-08-23 23:00:12,330 - wandb.wandb_agent - INFO - Agent received command: run
2023-08-23 23:00:12,331 - wandb.wandb_agent - INFO - Agent starting run with config:
	gamma: 0.099
	lr: 0.001
2023-08-23 23:00:12,333 - wandb.wandb_agent - INFO - About to run command: /usr/bin/env python /nas/ucb/mason/ethically-compliant-rl/CPO/train_cpo.py --gamma=0.099 --lr=0.001
wandb: Currently logged in as: mason-nakamura1. Use `wandb login --relogin` to force relogin
wandb: WARNING Ignored wandb.init() arg project when running a sweep.
wandb: WARNING Ignored wandb.init() arg entity when running a sweep.
2023-08-23 23:00:13,299 - wandb.wandb_agent - INFO - Cleaning up finished run: h3wa9i7n
2023-08-23 23:00:13,752 - wandb.wandb_agent - INFO - Agent received command: run
2023-08-23 23:00:13,752 - wandb.wandb_agent - INFO - Agent starting run with config:
	actor_lr: 0.0005
	critic_lr: 0.001
	gamma: 0.095
2023-08-23 23:00:13,755 - wandb.wandb_agent - INFO - About to run command: /usr/bin/env python /nas/ucb/mason/ethically-compliant-rl/CVPO/train_cvpo.py --actor_lr=0.0005 --critic_lr=0.001 --gamma=0.095
2023-08-23 16:00:14,798 - wandb.wandb_agent - INFO - Running runs: ['paawotay']
wandb: Currently logged in as: mason-nakamura1. Use `wandb login --relogin` to force relogin
wandb: WARNING Ignored wandb.init() arg project when running a sweep.
wandb: WARNING Ignored wandb.init() arg entity when running a sweep.
wandb: wandb version 0.15.8 is available!  To upgrade, please run:
wandb:  $ pip install wandb --upgrade
wandb: Tracking run with wandb version 0.14.2
wandb: Run data is saved locally in /nas/ucb/mason/ethically-compliant-rl/wandb/run-20230823_160012-paawotay
wandb: Run `wandb offline` to turn off syncing.
wandb: Syncing run fresh-sweep-10
wandb: ‚≠êÔ∏è View project at https://wandb.ai/ecrl/fast-safe-rl
wandb: üßπ View sweep at https://wandb.ai/ecrl/fast-safe-rl/sweeps/3qe1o66j
wandb: üöÄ View run at https://wandb.ai/ecrl/fast-safe-rl/runs/paawotay
wandb: wandb version 0.15.8 is available!  To upgrade, please run:
wandb:  $ pip install wandb --upgrade
wandb: Tracking run with wandb version 0.14.2
wandb: Run data is saved locally in /nas/ucb/mason/ethically-compliant-rl/wandb/run-20230823_230015-86hwy681
wandb: Run `wandb offline` to turn off syncing.
wandb: Syncing run twilight-sweep-10
wandb: ‚≠êÔ∏è View project at https://wandb.ai/ecrl/fast-safe-rl
wandb: üßπ View sweep at https://wandb.ai/ecrl/fast-safe-rl/sweeps/tea6k681
wandb: üöÄ View run at https://wandb.ai/ecrl/fast-safe-rl/runs/86hwy681
wandb: Currently logged in as: mason-nakamura1. Use `wandb login --relogin` to force relogin
wandb: WARNING Ignored wandb.init() arg project when running a sweep.
wandb: WARNING Ignored wandb.init() arg entity when running a sweep.
wandb: wandb version 0.15.8 is available!  To upgrade, please run:
wandb:  $ pip install wandb --upgrade
wandb: Tracking run with wandb version 0.14.2
wandb: Run data is saved locally in /nas/ucb/mason/ethically-compliant-rl/wandb/run-20230823_230016-1mfbt1dg
wandb: Run `wandb offline` to turn off syncing.
wandb: Syncing run lyric-sweep-11
wandb: ‚≠êÔ∏è View project at https://wandb.ai/ecrl/fast-safe-rl
wandb: üßπ View sweep at https://wandb.ai/ecrl/fast-safe-rl/sweeps/3qe1o66j
wandb: üöÄ View run at https://wandb.ai/ecrl/fast-safe-rl/runs/1mfbt1dg
2023-08-23 23:00:17,341 - wandb.wandb_agent - INFO - Running runs: ['86hwy681']
2023-08-23 16:00:17,401 - wandb.wandb_agent - INFO - Cleaning up finished run: 0nu523d4
2023-08-23 16:00:18,020 - wandb.wandb_agent - INFO - Agent received command: run
2023-08-23 16:00:18,020 - wandb.wandb_agent - INFO - Agent starting run with config:
	gamma: 0.095
	lr: 0.001
2023-08-23 16:00:18,023 - wandb.wandb_agent - INFO - About to run command: /usr/bin/env python /nas/ucb/mason/ethically-compliant-rl/CPO/train_cpo.py --gamma=0.095 --lr=0.001
2023-08-23 23:00:18,764 - wandb.wandb_agent - INFO - Running runs: ['1mfbt1dg']
[32;1mLogging data to logs/CPO-sweep-700epochs/parking-v0-cost-[5.0]/cpo_constraint_typelines_cost5.0_gamma0.099_step_per_epoch1000-92b2/progress.txt[0m
[32;1mLogging data to logs/CVPO-sweep-700epochs/parking-v0-cost-[5.0]/cvpo_constraint_typelines_cost5.0_gamma0.095_step_per_epoch1000-85e8/progress.txt[0m
wandb: Currently logged in as: mason-nakamura1. Use `wandb login --relogin` to force relogin
wandb: WARNING Ignored wandb.init() arg project when running a sweep.
wandb: WARNING Ignored wandb.init() arg entity when running a sweep.
wandb: wandb version 0.15.8 is available!  To upgrade, please run:
wandb:  $ pip install wandb --upgrade
wandb: Tracking run with wandb version 0.14.2
wandb: Run data is saved locally in /nas/ucb/mason/ethically-compliant-rl/wandb/run-20230823_160021-0ik4oy50
wandb: Run `wandb offline` to turn off syncing.
wandb: Syncing run wild-sweep-11
wandb: ‚≠êÔ∏è View project at https://wandb.ai/ecrl/fast-safe-rl
wandb: üßπ View sweep at https://wandb.ai/ecrl/fast-safe-rl/sweeps/tea6k681
wandb: üöÄ View run at https://wandb.ai/ecrl/fast-safe-rl/runs/0ik4oy50
[32;1mLogging data to logs/CVPO-sweep-700epochs/parking-v0-cost-[5.0]/cvpo_constraint_typelines_cost5.0_critic_lr0.0005_gamma0.095_step_per_epoch1000-9e7a/progress.txt[0m
2023-08-23 16:00:23,032 - wandb.wandb_agent - INFO - Running runs: ['0ik4oy50']
Epoch #1:   0%|          | 0/1000 [00:00<?, ?it/s]Epoch #1: 9803it [00:33, 295.45it/s]              Epoch #1: 9803it [00:33, 295.41it/s]
Traceback (most recent call last):
  File "/nas/ucb/mason/ethically-compliant-rl/CPO/train_cpo.py", line 237, in <module>
    train()
  File "/nas/ucb/mason/ethically-compliant-rl/.venv/lib/python3.9/site-packages/pyrallis/argparsing.py", line 158, in wrapper_inner
    response = fn(cfg, *args, **kwargs)
  File "/nas/ucb/mason/ethically-compliant-rl/CPO/train_cpo.py", line 202, in train
    agent.learn(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/agent/cpo_agent.py", line 229, in learn
    return super().learn(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/agent/base_agent.py", line 353, in learn
    for epoch, _epoch_stat, info in trainer:
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/base_trainer.py", line 208, in __next__
    self.policy_update_fn(stats_train)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/onpolicy.py", line 104, in policy_update_fn
    self.policy.update(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 361, in update
    batch = self.process_fn(batch, buffer, indices)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/cpo.py", line 130, in process_fn
    batch = self.compute_gae_returns(batch, buffer, indices, self._lambda)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 421, in compute_gae_returns
    metrics = self.get_metrics(batch)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 392, in get_metrics
    metrics = [batch.rew, cost[:, 0], cost[:, 1]]
IndexError: index 1 is out of bounds for axis 1 with size 1
wandb: Waiting for W&B process to finish... (failed 1). Press Control-C to abort syncing.
[32;1mLogging data to logs/CPO-sweep-700epochs/parking-v0-cost-[5.0]/cpo_constraint_typelines_cost5.0_gamma0.095_step_per_epoch1000-7bf3/progress.txt[0m
wandb: - 0.027 MB of 0.031 MB uploaded (0.000 MB deduped)wandb: \ 0.027 MB of 0.031 MB uploaded (0.000 MB deduped)wandb: üöÄ View run rich-sweep-9 at: https://wandb.ai/ecrl/fast-safe-rl/runs/iav5e5r3
wandb: Synced 5 W&B file(s), 0 media file(s), 0 artifact file(s) and 0 other file(s)
wandb: Find logs at: ./wandb/run-20230823_155940-iav5e5r3/logs
Exception in thread NetStatThr:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.9/threading.py", line 892, in run
    self._target(*self._args, **self._kwargs)
  File "/nas/ucb/mason/ethically-compliant-rl/.venv/lib/python3.9/site-packages/wandb/sdk/wandb_run.py", line 259, in check_network_status
    self._loop_check_status(
  File "/nas/ucb/mason/ethically-compliant-rl/.venv/lib/python3.9/site-packages/wandb/sdk/wandb_run.py", line 215, in _loop_check_status
    local_handle = request()
  File "/nas/ucb/mason/ethically-compliant-rl/.venv/lib/python3.9/site-packages/wandb/sdk/interface/interface.py", line 795, in deliver_network_status
    return self._deliver_network_status(status)
  File "/nas/ucb/mason/ethically-compliant-rl/.venv/lib/python3.9/site-packages/wandb/sdk/interface/interface_shared.py", line 601, in _deliver_network_status
    return self._deliver_record(record)
  File "/nas/ucb/mason/ethically-compliant-rl/.venv/lib/python3.9/site-packages/wandb/sdk/interface/interface_shared.py", line 560, in _deliver_record
    handle = mailbox._deliver_record(record, interface=self)
  File "/nas/ucb/mason/ethically-compliant-rl/.venv/lib/python3.9/site-packages/wandb/sdk/lib/mailbox.py", line 455, in _deliver_record
    interface._publish(record)
  File "/nas/ucb/mason/ethically-compliant-rl/.venv/lib/python3.9/site-packages/wandb/sdk/interface/interface_sock.py", line 51, in _publish
    self._sock_client.send_record_publish(record)
  File "/nas/ucb/mason/ethically-compliant-rl/.venv/lib/python3.9/site-packages/wandb/sdk/lib/sock_client.py", line 221, in send_record_publish
    self.send_server_request(server_req)
  File "/nas/ucb/mason/ethically-compliant-rl/.venv/lib/python3.9/site-packages/wandb/sdk/lib/sock_client.py", line 155, in send_server_request
    self._send_message(msg)
  File "/nas/ucb/mason/ethically-compliant-rl/.venv/lib/python3.9/site-packages/wandb/sdk/lib/sock_client.py", line 152, in _send_message
    self._sendall_with_error_handle(header + data)
  File "/nas/ucb/mason/ethically-compliant-rl/.venv/lib/python3.9/site-packages/wandb/sdk/lib/sock_client.py", line 130, in _sendall_with_error_handle
    sent = self._sock.send(data)
BrokenPipeError: [Errno 32] Broken pipe
2023-08-23 16:00:33,439 - wandb.wandb_agent - INFO - Cleaning up finished run: iav5e5r3
2023-08-23 16:00:33,859 - wandb.wandb_agent - INFO - Agent received command: run
2023-08-23 16:00:33,859 - wandb.wandb_agent - INFO - Agent starting run with config:
	gamma: 0.095
	lr: 0.001
2023-08-23 16:00:33,863 - wandb.wandb_agent - INFO - About to run command: /usr/bin/env python /nas/ucb/mason/ethically-compliant-rl/CPO/train_cpo.py --gamma=0.095 --lr=0.001
wandb: Currently logged in as: mason-nakamura1. Use `wandb login --relogin` to force relogin
wandb: WARNING Ignored wandb.init() arg project when running a sweep.
wandb: WARNING Ignored wandb.init() arg entity when running a sweep.
wandb: wandb version 0.15.8 is available!  To upgrade, please run:
wandb:  $ pip install wandb --upgrade
wandb: Tracking run with wandb version 0.14.2
wandb: Run data is saved locally in /nas/ucb/mason/ethically-compliant-rl/wandb/run-20230823_160036-mvjx744v
wandb: Run `wandb offline` to turn off syncing.
wandb: Syncing run clear-sweep-12
wandb: ‚≠êÔ∏è View project at https://wandb.ai/ecrl/fast-safe-rl
wandb: üßπ View sweep at https://wandb.ai/ecrl/fast-safe-rl/sweeps/tea6k681
wandb: üöÄ View run at https://wandb.ai/ecrl/fast-safe-rl/runs/mvjx744v
2023-08-23 16:00:38,874 - wandb.wandb_agent - INFO - Running runs: ['mvjx744v']
[32;1mLogging data to logs/CPO-sweep-700epochs/parking-v0-cost-[5.0]/cpo_constraint_typelines_cost5.0_gamma0.095_step_per_epoch1000-d119/progress.txt[0m
Epoch #1:   0%|          | 0/1000 [00:00<?, ?it/s]Epoch #1: 9205it [00:43, 212.62it/s]              Epoch #1: 9205it [00:43, 212.60it/s]
Traceback (most recent call last):
  File "/nas/ucb/mason/ethically-compliant-rl/CVPO/train_cvpo.py", line 246, in <module>
    train()
  File "/nas/ucb/mason/ethically-compliant-rl/.venv/lib/python3.9/site-packages/pyrallis/argparsing.py", line 158, in wrapper_inner
    response = fn(cfg, *args, **kwargs)
  File "/nas/ucb/mason/ethically-compliant-rl/CVPO/train_cvpo.py", line 211, in train
    agent.learn(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/agent/base_agent.py", line 236, in learn
    for epoch, _epoch_stat, info in trainer:
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/base_trainer.py", line 208, in __next__
    self.policy_update_fn(stats_train)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/offpolicy.py", line 106, in policy_update_fn
    self.policy.update(self.batch_size, self.train_collector.buffer)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 361, in update
    batch = self.process_fn(batch, buffer, indices)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/cvpo.py", line 216, in process_fn
    batch = self.compute_nstep_returns(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 494, in compute_nstep_returns
    metrics = self.get_metrics(buffer)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 392, in get_metrics
    metrics = [batch.rew, cost[:, 0], cost[:, 1]]
IndexError: index 1 is out of bounds for axis 1 with size 1
Traceback (most recent call last):
  File "/nas/ucb/mason/ethically-compliant-rl/CVPO/train_cvpo.py", line 246, in <module>
    train()
  File "/nas/ucb/mason/ethically-compliant-rl/.venv/lib/python3.9/site-packages/pyrallis/argparsing.py", line 158, in wrapper_inner
    response = fn(cfg, *args, **kwargs)
  File "/nas/ucb/mason/ethically-compliant-rl/CVPO/train_cvpo.py", line 211, in train
    agent.learn(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/agent/base_agent.py", line 236, in learn
    for epoch, _epoch_stat, info in trainer:
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/base_trainer.py", line 208, in __next__
    self.policy_update_fn(stats_train)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/offpolicy.py", line 106, in policy_update_fn
    self.policy.update(self.batch_size, self.train_collector.buffer)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 361, in update
    batch = self.process_fn(batch, buffer, indices)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/cvpo.py", line 216, in process_fn
    batch = self.compute_nstep_returns(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 494, in compute_nstep_returns
    metrics = self.get_metrics(buffer)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 392, in get_metrics
    metrics = [batch.rew, cost[:, 0], cost[:, 1]]
IndexError: index 1 is out of bounds for axis 1 with size 1
wandb: Waiting for W&B process to finish... (failed 1). Press Control-C to abort syncing.
Epoch #1:   0%|          | 0/1000 [00:00<?, ?it/s]Epoch #1: 9542it [00:30, 311.48it/s]              Epoch #1: 9542it [00:30, 311.44it/s]
Traceback (most recent call last):
  File "/nas/ucb/mason/ethically-compliant-rl/CPO/train_cpo.py", line 237, in <module>
    train()
  File "/nas/ucb/mason/ethically-compliant-rl/.venv/lib/python3.9/site-packages/pyrallis/argparsing.py", line 158, in wrapper_inner
    response = fn(cfg, *args, **kwargs)
  File "/nas/ucb/mason/ethically-compliant-rl/CPO/train_cpo.py", line 202, in train
    agent.learn(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/agent/cpo_agent.py", line 229, in learn
    return super().learn(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/agent/base_agent.py", line 353, in learn
    for epoch, _epoch_stat, info in trainer:
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/base_trainer.py", line 208, in __next__
    self.policy_update_fn(stats_train)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/onpolicy.py", line 104, in policy_update_fn
    self.policy.update(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 361, in update
    batch = self.process_fn(batch, buffer, indices)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/cpo.py", line 130, in process_fn
    batch = self.compute_gae_returns(batch, buffer, indices, self._lambda)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 421, in compute_gae_returns
    metrics = self.get_metrics(batch)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 392, in get_metrics
    metrics = [batch.rew, cost[:, 0], cost[:, 1]]
IndexError: index 1 is out of bounds for axis 1 with size 1
wandb: Waiting for W&B process to finish... (failed 1). Press Control-C to abort syncing.
wandb: üöÄ View run celestial-sweep-9 at: https://wandb.ai/ecrl/fast-safe-rl/runs/qjjv7wrw
wandb: Synced 5 W&B file(s), 0 media file(s), 0 artifact file(s) and 0 other file(s)
wandb: Find logs at: ./wandb/run-20230823_160001-qjjv7wrw/logs
wandb: - 0.027 MB of 0.027 MB uploaded (0.000 MB deduped)wandb: \ 0.028 MB of 0.031 MB uploaded (0.000 MB deduped)wandb: üöÄ View run twilight-sweep-10 at: https://wandb.ai/ecrl/fast-safe-rl/runs/86hwy681
wandb: Synced 5 W&B file(s), 0 media file(s), 0 artifact file(s) and 0 other file(s)
wandb: Find logs at: ./wandb/run-20230823_230015-86hwy681/logs
2023-08-23 16:00:58,849 - wandb.wandb_agent - INFO - Cleaning up finished run: qjjv7wrw
Epoch #1:   0%|          | 0/1000 [00:00<?, ?it/s]Epoch #1: 9465it [00:36, 260.10it/s]              Epoch #1: 9465it [00:36, 260.08it/s]
Traceback (most recent call last):
  File "/nas/ucb/mason/ethically-compliant-rl/CVPO/train_cvpo.py", line 246, in <module>
    train()
  File "/nas/ucb/mason/ethically-compliant-rl/.venv/lib/python3.9/site-packages/pyrallis/argparsing.py", line 158, in wrapper_inner
    response = fn(cfg, *args, **kwargs)
  File "/nas/ucb/mason/ethically-compliant-rl/CVPO/train_cvpo.py", line 211, in train
    agent.learn(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/agent/base_agent.py", line 236, in learn
    for epoch, _epoch_stat, info in trainer:
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/base_trainer.py", line 208, in __next__
    self.policy_update_fn(stats_train)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/offpolicy.py", line 106, in policy_update_fn
    self.policy.update(self.batch_size, self.train_collector.buffer)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 361, in update
    batch = self.process_fn(batch, buffer, indices)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/cvpo.py", line 216, in process_fn
    batch = self.compute_nstep_returns(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 494, in compute_nstep_returns
    metrics = self.get_metrics(buffer)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 392, in get_metrics
    metrics = [batch.rew, cost[:, 0], cost[:, 1]]
IndexError: index 1 is out of bounds for axis 1 with size 1
Traceback (most recent call last):
  File "/nas/ucb/mason/ethically-compliant-rl/CVPO/train_cvpo.py", line 246, in <module>
    train()
  File "/nas/ucb/mason/ethically-compliant-rl/.venv/lib/python3.9/site-packages/pyrallis/argparsing.py", line 158, in wrapper_inner
    response = fn(cfg, *args, **kwargs)
  File "/nas/ucb/mason/ethically-compliant-rl/CVPO/train_cvpo.py", line 211, in train
    agent.learn(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/agent/base_agent.py", line 236, in learn
    for epoch, _epoch_stat, info in trainer:
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/base_trainer.py", line 208, in __next__
    self.policy_update_fn(stats_train)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/offpolicy.py", line 106, in policy_update_fn
    self.policy.update(self.batch_size, self.train_collector.buffer)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 361, in update
    batch = self.process_fn(batch, buffer, indices)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/cvpo.py", line 216, in process_fn
    batch = self.compute_nstep_returns(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 494, in compute_nstep_returns
    metrics = self.get_metrics(buffer)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 392, in get_metrics
    metrics = [batch.rew, cost[:, 0], cost[:, 1]]
IndexError: index 1 is out of bounds for axis 1 with size 1
wandb: Waiting for W&B process to finish... (failed 1). Press Control-C to abort syncing.
2023-08-23 16:00:59,450 - wandb.wandb_agent - INFO - Agent received command: run
2023-08-23 16:00:59,450 - wandb.wandb_agent - INFO - Agent starting run with config:
	actor_lr: 0.0005
	critic_lr: 0.001
	gamma: 0.099
2023-08-23 16:00:59,453 - wandb.wandb_agent - INFO - About to run command: /usr/bin/env python /nas/ucb/mason/ethically-compliant-rl/CVPO/train_cvpo.py --actor_lr=0.0005 --critic_lr=0.001 --gamma=0.099
wandb: - 0.027 MB of 0.027 MB uploaded (0.000 MB deduped)wandb: \ 0.027 MB of 0.032 MB uploaded (0.000 MB deduped)wandb: | 0.033 MB of 0.033 MB uploaded (0.000 MB deduped)wandb: üöÄ View run lyric-sweep-11 at: https://wandb.ai/ecrl/fast-safe-rl/runs/1mfbt1dg
wandb: Synced 5 W&B file(s), 0 media file(s), 0 artifact file(s) and 0 other file(s)
wandb: Find logs at: ./wandb/run-20230823_230016-1mfbt1dg/logs
wandb: Currently logged in as: mason-nakamura1. Use `wandb login --relogin` to force relogin
wandb: WARNING Ignored wandb.init() arg project when running a sweep.
wandb: WARNING Ignored wandb.init() arg entity when running a sweep.
2023-08-23 23:01:03,159 - wandb.wandb_agent - INFO - Cleaning up finished run: 86hwy681
wandb: wandb version 0.15.8 is available!  To upgrade, please run:
wandb:  $ pip install wandb --upgrade
wandb: Tracking run with wandb version 0.14.2
wandb: Run data is saved locally in /nas/ucb/mason/ethically-compliant-rl/wandb/run-20230823_160103-y463p9fp
wandb: Run `wandb offline` to turn off syncing.
wandb: Syncing run faithful-sweep-12
wandb: ‚≠êÔ∏è View project at https://wandb.ai/ecrl/fast-safe-rl
wandb: üßπ View sweep at https://wandb.ai/ecrl/fast-safe-rl/sweeps/3qe1o66j
wandb: üöÄ View run at https://wandb.ai/ecrl/fast-safe-rl/runs/y463p9fp
2023-08-23 23:01:03,684 - wandb.wandb_agent - INFO - Agent received command: run
2023-08-23 23:01:03,685 - wandb.wandb_agent - INFO - Agent starting run with config:
	gamma: 0.095
	lr: 0.0005
2023-08-23 23:01:03,687 - wandb.wandb_agent - INFO - About to run command: /usr/bin/env python /nas/ucb/mason/ethically-compliant-rl/CPO/train_cpo.py --gamma=0.095 --lr=0.0005
2023-08-23 16:01:04,463 - wandb.wandb_agent - INFO - Running runs: ['y463p9fp']
Epoch #1:   0%|          | 0/1000 [00:00<?, ?it/s]Epoch #1: 9480it [00:40, 234.45it/s]              Epoch #1: 9480it [00:40, 234.44it/s]
Traceback (most recent call last):
  File "/nas/ucb/mason/ethically-compliant-rl/CVPO/train_cvpo.py", line 246, in <module>
    train()
  File "/nas/ucb/mason/ethically-compliant-rl/.venv/lib/python3.9/site-packages/pyrallis/argparsing.py", line 158, in wrapper_inner
    response = fn(cfg, *args, **kwargs)
  File "/nas/ucb/mason/ethically-compliant-rl/CVPO/train_cvpo.py", line 211, in train
    agent.learn(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/agent/base_agent.py", line 236, in learn
    for epoch, _epoch_stat, info in trainer:
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/base_trainer.py", line 208, in __next__
    self.policy_update_fn(stats_train)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/offpolicy.py", line 106, in policy_update_fn
    self.policy.update(self.batch_size, self.train_collector.buffer)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 361, in update
    batch = self.process_fn(batch, buffer, indices)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/cvpo.py", line 216, in process_fn
    batch = self.compute_nstep_returns(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 494, in compute_nstep_returns
    metrics = self.get_metrics(buffer)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 392, in get_metrics
    metrics = [batch.rew, cost[:, 0], cost[:, 1]]
IndexError: index 1 is out of bounds for axis 1 with size 1
Traceback (most recent call last):
  File "/nas/ucb/mason/ethically-compliant-rl/CVPO/train_cvpo.py", line 246, in <module>
    train()
  File "/nas/ucb/mason/ethically-compliant-rl/.venv/lib/python3.9/site-packages/pyrallis/argparsing.py", line 158, in wrapper_inner
    response = fn(cfg, *args, **kwargs)
  File "/nas/ucb/mason/ethically-compliant-rl/CVPO/train_cvpo.py", line 211, in train
    agent.learn(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/agent/base_agent.py", line 236, in learn
    for epoch, _epoch_stat, info in trainer:
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/base_trainer.py", line 208, in __next__
    self.policy_update_fn(stats_train)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/offpolicy.py", line 106, in policy_update_fn
    self.policy.update(self.batch_size, self.train_collector.buffer)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 361, in update
    batch = self.process_fn(batch, buffer, indices)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/cvpo.py", line 216, in process_fn
    batch = self.compute_nstep_returns(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 494, in compute_nstep_returns
    metrics = self.get_metrics(buffer)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 392, in get_metrics
    metrics = [batch.rew, cost[:, 0], cost[:, 1]]
IndexError: index 1 is out of bounds for axis 1 with size 1
wandb: Waiting for W&B process to finish... (failed 1). Press Control-C to abort syncing.
wandb: Currently logged in as: mason-nakamura1. Use `wandb login --relogin` to force relogin
wandb: WARNING Ignored wandb.init() arg project when running a sweep.
wandb: WARNING Ignored wandb.init() arg entity when running a sweep.
wandb: wandb version 0.15.8 is available!  To upgrade, please run:
wandb:  $ pip install wandb --upgrade
wandb: Tracking run with wandb version 0.14.2
wandb: Run data is saved locally in /nas/ucb/mason/ethically-compliant-rl/wandb/run-20230823_230106-dekd7aqn
wandb: Run `wandb offline` to turn off syncing.
wandb: Syncing run tough-sweep-13
wandb: ‚≠êÔ∏è View project at https://wandb.ai/ecrl/fast-safe-rl
wandb: üßπ View sweep at https://wandb.ai/ecrl/fast-safe-rl/sweeps/tea6k681
wandb: üöÄ View run at https://wandb.ai/ecrl/fast-safe-rl/runs/dekd7aqn
[32;1mLogging data to logs/CVPO-sweep-700epochs/parking-v0-cost-[5.0]/cvpo_constraint_typelines_cost5.0_gamma0.099_step_per_epoch1000-65c9/progress.txt[0m
2023-08-23 23:01:08,696 - wandb.wandb_agent - INFO - Running runs: ['dekd7aqn']
Epoch #1:   0%|          | 0/1000 [00:00<?, ?it/s]Epoch #1: 10959it [00:38, 282.56it/s]             Epoch #1: 10959it [00:38, 282.53it/s]
Traceback (most recent call last):
  File "/nas/ucb/mason/ethically-compliant-rl/CPO/train_cpo.py", line 237, in <module>
    train()
  File "/nas/ucb/mason/ethically-compliant-rl/.venv/lib/python3.9/site-packages/pyrallis/argparsing.py", line 158, in wrapper_inner
    response = fn(cfg, *args, **kwargs)
  File "/nas/ucb/mason/ethically-compliant-rl/CPO/train_cpo.py", line 202, in train
    agent.learn(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/agent/cpo_agent.py", line 229, in learn
    return super().learn(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/agent/base_agent.py", line 353, in learn
    for epoch, _epoch_stat, info in trainer:
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/base_trainer.py", line 208, in __next__
    self.policy_update_fn(stats_train)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/onpolicy.py", line 104, in policy_update_fn
    self.policy.update(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 361, in update
    batch = self.process_fn(batch, buffer, indices)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/cpo.py", line 130, in process_fn
    batch = self.compute_gae_returns(batch, buffer, indices, self._lambda)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 421, in compute_gae_returns
    metrics = self.get_metrics(batch)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 392, in get_metrics
    metrics = [batch.rew, cost[:, 0], cost[:, 1]]
IndexError: index 1 is out of bounds for axis 1 with size 1
wandb: Waiting for W&B process to finish... (failed 1). Press Control-C to abort syncing.
2023-08-23 23:01:09,645 - wandb.wandb_agent - INFO - Cleaning up finished run: 1mfbt1dg
2023-08-23 23:01:10,168 - wandb.wandb_agent - INFO - Agent received command: run
2023-08-23 23:01:10,169 - wandb.wandb_agent - INFO - Agent starting run with config:
	actor_lr: 0.001
	critic_lr: 0.001
	gamma: 0.099
2023-08-23 23:01:10,171 - wandb.wandb_agent - INFO - About to run command: /usr/bin/env python /nas/ucb/mason/ethically-compliant-rl/CVPO/train_cvpo.py --actor_lr=0.001 --critic_lr=0.001 --gamma=0.099
wandb: üöÄ View run fresh-sweep-10 at: https://wandb.ai/ecrl/fast-safe-rl/runs/paawotay
wandb: Synced 5 W&B file(s), 0 media file(s), 0 artifact file(s) and 0 other file(s)
wandb: Find logs at: ./wandb/run-20230823_160012-paawotay/logs
[32;1mLogging data to logs/CPO-sweep-700epochs/parking-v0-cost-[5.0]/cpo_constraint_typelines_cost5.0_gamma0.095_lr0.0005_step_per_epoch1000-d397/progress.txt[0m
wandb: - 0.027 MB of 0.027 MB uploaded (0.000 MB deduped)wandb: \ 0.031 MB of 0.031 MB uploaded (0.000 MB deduped)wandb: üöÄ View run wild-sweep-11 at: https://wandb.ai/ecrl/fast-safe-rl/runs/0ik4oy50
wandb: Synced 5 W&B file(s), 0 media file(s), 0 artifact file(s) and 0 other file(s)
wandb: Find logs at: ./wandb/run-20230823_160021-0ik4oy50/logs
wandb: Currently logged in as: mason-nakamura1. Use `wandb login --relogin` to force relogin
wandb: WARNING Ignored wandb.init() arg project when running a sweep.
wandb: WARNING Ignored wandb.init() arg entity when running a sweep.
wandb: wandb version 0.15.8 is available!  To upgrade, please run:
wandb:  $ pip install wandb --upgrade
wandb: Tracking run with wandb version 0.14.2
wandb: Run data is saved locally in /nas/ucb/mason/ethically-compliant-rl/wandb/run-20230823_230113-rt4v81cz
wandb: Run `wandb offline` to turn off syncing.
wandb: Syncing run visionary-sweep-13
wandb: ‚≠êÔ∏è View project at https://wandb.ai/ecrl/fast-safe-rl
wandb: üßπ View sweep at https://wandb.ai/ecrl/fast-safe-rl/sweeps/3qe1o66j
wandb: üöÄ View run at https://wandb.ai/ecrl/fast-safe-rl/runs/rt4v81cz
2023-08-23 16:01:14,140 - wandb.wandb_agent - INFO - Cleaning up finished run: 0ik4oy50
2023-08-23 16:01:14,580 - wandb.wandb_agent - INFO - Agent received command: run
2023-08-23 16:01:14,581 - wandb.wandb_agent - INFO - Agent starting run with config:
	gamma: 0.099
	lr: 0.0005
2023-08-23 16:01:14,583 - wandb.wandb_agent - INFO - About to run command: /usr/bin/env python /nas/ucb/mason/ethically-compliant-rl/CPO/train_cpo.py --gamma=0.099 --lr=0.0005
2023-08-23 23:01:15,177 - wandb.wandb_agent - INFO - Running runs: ['rt4v81cz']
2023-08-23 16:01:15,842 - wandb.wandb_agent - INFO - Cleaning up finished run: paawotay
2023-08-23 16:01:16,403 - wandb.wandb_agent - INFO - Agent received command: run
2023-08-23 16:01:16,403 - wandb.wandb_agent - INFO - Agent starting run with config:
	actor_lr: 0.001
	critic_lr: 0.001
	gamma: 0.095
2023-08-23 16:01:16,406 - wandb.wandb_agent - INFO - About to run command: /usr/bin/env python /nas/ucb/mason/ethically-compliant-rl/CVPO/train_cvpo.py --actor_lr=0.001 --critic_lr=0.001 --gamma=0.095
[32;1mLogging data to logs/CVPO-sweep-700epochs/parking-v0-cost-[5.0]/cvpo_actor_lr0.001_constraint_typelines_cost5.0_gamma0.099_step_per_epoch1000-42bc/progress.txt[0m
Epoch #1:   0%|          | 0/1000 [00:00<?, ?it/s]Epoch #1: 9555it [00:32, 296.56it/s]              Epoch #1: 9555it [00:32, 296.52it/s]
Traceback (most recent call last):
  File "/nas/ucb/mason/ethically-compliant-rl/CPO/train_cpo.py", line 237, in <module>
    train()
  File "/nas/ucb/mason/ethically-compliant-rl/.venv/lib/python3.9/site-packages/pyrallis/argparsing.py", line 158, in wrapper_inner
    response = fn(cfg, *args, **kwargs)
  File "/nas/ucb/mason/ethically-compliant-rl/CPO/train_cpo.py", line 202, in train
    agent.learn(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/agent/cpo_agent.py", line 229, in learn
    return super().learn(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/agent/base_agent.py", line 353, in learn
    for epoch, _epoch_stat, info in trainer:
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/base_trainer.py", line 208, in __next__
    self.policy_update_fn(stats_train)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/onpolicy.py", line 104, in policy_update_fn
    self.policy.update(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 361, in update
    batch = self.process_fn(batch, buffer, indices)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/cpo.py", line 130, in process_fn
    batch = self.compute_gae_returns(batch, buffer, indices, self._lambda)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 421, in compute_gae_returns
    metrics = self.get_metrics(batch)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 392, in get_metrics
    metrics = [batch.rew, cost[:, 0], cost[:, 1]]
IndexError: index 1 is out of bounds for axis 1 with size 1
wandb: Waiting for W&B process to finish... (failed 1). Press Control-C to abort syncing.
wandb: Currently logged in as: mason-nakamura1. Use `wandb login --relogin` to force relogin
wandb: WARNING Ignored wandb.init() arg project when running a sweep.
wandb: WARNING Ignored wandb.init() arg entity when running a sweep.
wandb: wandb version 0.15.8 is available!  To upgrade, please run:
wandb:  $ pip install wandb --upgrade
wandb: Tracking run with wandb version 0.14.2
wandb: Run data is saved locally in /nas/ucb/mason/ethically-compliant-rl/wandb/run-20230823_160117-of9m5b26
wandb: Run `wandb offline` to turn off syncing.
wandb: Syncing run golden-sweep-14
wandb: ‚≠êÔ∏è View project at https://wandb.ai/ecrl/fast-safe-rl
wandb: üßπ View sweep at https://wandb.ai/ecrl/fast-safe-rl/sweeps/tea6k681
wandb: üöÄ View run at https://wandb.ai/ecrl/fast-safe-rl/runs/of9m5b26
2023-08-23 16:01:19,592 - wandb.wandb_agent - INFO - Running runs: ['of9m5b26']
wandb: Currently logged in as: mason-nakamura1. Use `wandb login --relogin` to force relogin
wandb: WARNING Ignored wandb.init() arg project when running a sweep.
wandb: WARNING Ignored wandb.init() arg entity when running a sweep.
wandb: - 0.027 MB of 0.027 MB uploaded (0.000 MB deduped)wandb: \ 0.027 MB of 0.031 MB uploaded (0.000 MB deduped)wandb: | 0.031 MB of 0.031 MB uploaded (0.000 MB deduped)wandb: üöÄ View run clear-sweep-12 at: https://wandb.ai/ecrl/fast-safe-rl/runs/mvjx744v
wandb: Synced 5 W&B file(s), 0 media file(s), 0 artifact file(s) and 0 other file(s)
wandb: Find logs at: ./wandb/run-20230823_160036-mvjx744v/logs
2023-08-23 16:01:21,418 - wandb.wandb_agent - INFO - Running runs: ['fsxb61zg']
Exception in thread NetStatThr:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.9/threading.py", line 892, in run
    self._target(*self._args, **self._kwargs)
  File "/nas/ucb/mason/ethically-compliant-rl/.venv/lib/python3.9/site-packages/wandb/sdk/wandb_run.py", line 259, in check_network_status
    self._loop_check_status(
  File "/nas/ucb/mason/ethically-compliant-rl/.venv/lib/python3.9/site-packages/wandb/sdk/wandb_run.py", line 215, in _loop_check_status
    local_handle = request()
  File "/nas/ucb/mason/ethically-compliant-rl/.venv/lib/python3.9/site-packages/wandb/sdk/interface/interface.py", line 795, in deliver_network_status
    return self._deliver_network_status(status)
  File "/nas/ucb/mason/ethically-compliant-rl/.venv/lib/python3.9/site-packages/wandb/sdk/interface/interface_shared.py", line 601, in _deliver_network_status
    return self._deliver_record(record)
  File "/nas/ucb/mason/ethically-compliant-rl/.venv/lib/python3.9/site-packages/wandb/sdk/interface/interface_shared.py", line 560, in _deliver_record
    handle = mailbox._deliver_record(record, interface=self)
  File "/nas/ucb/mason/ethically-compliant-rl/.venv/lib/python3.9/site-packages/wandb/sdk/lib/mailbox.py", line 455, in _deliver_record
    interface._publish(record)
  File "/nas/ucb/mason/ethically-compliant-rl/.venv/lib/python3.9/site-packages/wandb/sdk/interface/interface_sock.py", line 51, in _publish
    self._sock_client.send_record_publish(record)
  File "/nas/ucb/mason/ethically-compliant-rl/.venv/lib/python3.9/site-packages/wandb/sdk/lib/sock_client.py", line 221, in send_record_publish
    self.send_server_request(server_req)
  File "/nas/ucb/mason/ethically-compliant-rl/.venv/lib/python3.9/site-packages/wandb/sdk/lib/sock_client.py", line 155, in send_server_request
    self._send_message(msg)
  File "/nas/ucb/mason/ethically-compliant-rl/.venv/lib/python3.9/site-packages/wandb/sdk/lib/sock_client.py", line 152, in _send_message
    self._sendall_with_error_handle(header + data)
  File "/nas/ucb/mason/ethically-compliant-rl/.venv/lib/python3.9/site-packages/wandb/sdk/lib/sock_client.py", line 130, in _sendall_with_error_handle
    sent = self._sock.send(data)
BrokenPipeError: [Errno 32] Broken pipe
[32;1mLogging data to logs/CPO-sweep-700epochs/parking-v0-cost-[5.0]/cpo_constraint_typelines_cost5.0_gamma0.099_lr0.0005_step_per_epoch1000-438c/progress.txt[0m
wandb: wandb version 0.15.8 is available!  To upgrade, please run:
wandb:  $ pip install wandb --upgrade
wandb: Tracking run with wandb version 0.14.2
wandb: Run data is saved locally in /nas/ucb/mason/ethically-compliant-rl/wandb/run-20230823_160120-fsxb61zg
wandb: Run `wandb offline` to turn off syncing.
wandb: Syncing run good-sweep-14
wandb: ‚≠êÔ∏è View project at https://wandb.ai/ecrl/fast-safe-rl
wandb: üßπ View sweep at https://wandb.ai/ecrl/fast-safe-rl/sweeps/3qe1o66j
wandb: üöÄ View run at https://wandb.ai/ecrl/fast-safe-rl/runs/fsxb61zg
2023-08-23 16:01:24,687 - wandb.wandb_agent - INFO - Cleaning up finished run: mvjx744v
2023-08-23 16:01:25,188 - wandb.wandb_agent - INFO - Agent received command: run
2023-08-23 16:01:25,188 - wandb.wandb_agent - INFO - Agent starting run with config:
	gamma: 0.099
	lr: 0.0005
2023-08-23 16:01:25,191 - wandb.wandb_agent - INFO - About to run command: /usr/bin/env python /nas/ucb/mason/ethically-compliant-rl/CPO/train_cpo.py --gamma=0.099 --lr=0.0005
wandb: Currently logged in as: mason-nakamura1. Use `wandb login --relogin` to force relogin
wandb: WARNING Ignored wandb.init() arg project when running a sweep.
wandb: WARNING Ignored wandb.init() arg entity when running a sweep.
wandb: wandb version 0.15.8 is available!  To upgrade, please run:
wandb:  $ pip install wandb --upgrade
wandb: Tracking run with wandb version 0.14.2
wandb: Run data is saved locally in /nas/ucb/mason/ethically-compliant-rl/wandb/run-20230823_160127-1obdrl1e
wandb: Run `wandb offline` to turn off syncing.
wandb: Syncing run charmed-sweep-15
wandb: ‚≠êÔ∏è View project at https://wandb.ai/ecrl/fast-safe-rl
wandb: üßπ View sweep at https://wandb.ai/ecrl/fast-safe-rl/sweeps/tea6k681
wandb: üöÄ View run at https://wandb.ai/ecrl/fast-safe-rl/runs/1obdrl1e
2023-08-23 16:01:30,201 - wandb.wandb_agent - INFO - Running runs: ['1obdrl1e']
[32;1mLogging data to logs/CVPO-sweep-700epochs/parking-v0-cost-[5.0]/cvpo_actor_lr0.001_constraint_typelines_cost5.0_gamma0.095_step_per_epoch1000-a5b0/progress.txt[0m
[32;1mLogging data to logs/CPO-sweep-700epochs/parking-v0-cost-[5.0]/cpo_constraint_typelines_cost5.0_gamma0.099_lr0.0005_step_per_epoch1000-14c2/progress.txt[0m
Epoch #1:   0%|          | 0/1000 [00:00<?, ?it/s]Epoch #1: 9889it [00:29, 331.06it/s]              Epoch #1: 9889it [00:29, 331.01it/s]
Traceback (most recent call last):
  File "/nas/ucb/mason/ethically-compliant-rl/CPO/train_cpo.py", line 237, in <module>
    train()
  File "/nas/ucb/mason/ethically-compliant-rl/.venv/lib/python3.9/site-packages/pyrallis/argparsing.py", line 158, in wrapper_inner
    response = fn(cfg, *args, **kwargs)
  File "/nas/ucb/mason/ethically-compliant-rl/CPO/train_cpo.py", line 202, in train
    agent.learn(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/agent/cpo_agent.py", line 229, in learn
    return super().learn(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/agent/base_agent.py", line 353, in learn
    for epoch, _epoch_stat, info in trainer:
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/base_trainer.py", line 208, in __next__
    self.policy_update_fn(stats_train)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/onpolicy.py", line 104, in policy_update_fn
    self.policy.update(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 361, in update
    batch = self.process_fn(batch, buffer, indices)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/cpo.py", line 130, in process_fn
    batch = self.compute_gae_returns(batch, buffer, indices, self._lambda)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 421, in compute_gae_returns
    metrics = self.get_metrics(batch)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 392, in get_metrics
    metrics = [batch.rew, cost[:, 0], cost[:, 1]]
IndexError: index 1 is out of bounds for axis 1 with size 1
wandb: Waiting for W&B process to finish... (failed 1). Press Control-C to abort syncing.
wandb: üöÄ View run tough-sweep-13 at: https://wandb.ai/ecrl/fast-safe-rl/runs/dekd7aqn
wandb: Synced 5 W&B file(s), 0 media file(s), 0 artifact file(s) and 0 other file(s)
wandb: Find logs at: ./wandb/run-20230823_230106-dekd7aqn/logs
Epoch #1:   0%|          | 0/1000 [00:00<?, ?it/s]Epoch #1: 8904it [00:43, 203.15it/s]              Epoch #1: 8904it [00:43, 203.13it/s]
Traceback (most recent call last):
  File "/nas/ucb/mason/ethically-compliant-rl/CVPO/train_cvpo.py", line 246, in <module>
    train()
  File "/nas/ucb/mason/ethically-compliant-rl/.venv/lib/python3.9/site-packages/pyrallis/argparsing.py", line 158, in wrapper_inner
    response = fn(cfg, *args, **kwargs)
  File "/nas/ucb/mason/ethically-compliant-rl/CVPO/train_cvpo.py", line 211, in train
    agent.learn(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/agent/base_agent.py", line 236, in learn
    for epoch, _epoch_stat, info in trainer:
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/base_trainer.py", line 208, in __next__
    self.policy_update_fn(stats_train)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/offpolicy.py", line 106, in policy_update_fn
    self.policy.update(self.batch_size, self.train_collector.buffer)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 361, in update
    batch = self.process_fn(batch, buffer, indices)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/cvpo.py", line 216, in process_fn
    batch = self.compute_nstep_returns(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 494, in compute_nstep_returns
    metrics = self.get_metrics(buffer)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 392, in get_metrics
    metrics = [batch.rew, cost[:, 0], cost[:, 1]]
IndexError: index 1 is out of bounds for axis 1 with size 1
Traceback (most recent call last):
  File "/nas/ucb/mason/ethically-compliant-rl/CVPO/train_cvpo.py", line 246, in <module>
    train()
  File "/nas/ucb/mason/ethically-compliant-rl/.venv/lib/python3.9/site-packages/pyrallis/argparsing.py", line 158, in wrapper_inner
    response = fn(cfg, *args, **kwargs)
  File "/nas/ucb/mason/ethically-compliant-rl/CVPO/train_cvpo.py", line 211, in train
    agent.learn(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/agent/base_agent.py", line 236, in learn
    for epoch, _epoch_stat, info in trainer:
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/base_trainer.py", line 208, in __next__
    self.policy_update_fn(stats_train)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/offpolicy.py", line 106, in policy_update_fn
    self.policy.update(self.batch_size, self.train_collector.buffer)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 361, in update
    batch = self.process_fn(batch, buffer, indices)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/cvpo.py", line 216, in process_fn
    batch = self.compute_nstep_returns(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 494, in compute_nstep_returns
    metrics = self.get_metrics(buffer)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 392, in get_metrics
    metrics = [batch.rew, cost[:, 0], cost[:, 1]]
IndexError: index 1 is out of bounds for axis 1 with size 1
wandb: Waiting for W&B process to finish... (failed 1). Press Control-C to abort syncing.
2023-08-23 23:01:54,433 - wandb.wandb_agent - ERROR - Detected 5 failed runs in a row, shutting down.
2023-08-23 23:01:54,434 - wandb.wandb_agent - INFO - To change this value set WANDB_AGENT_MAX_INITIAL_FAILURES=val
wandb: Terminating and syncing runs. Press ctrl-c to kill.
Epoch #1:   0%|          | 0/1000 [00:00<?, ?it/s]Epoch #1: 9929it [00:37, 265.79it/s]              Epoch #1: 9929it [00:37, 265.77it/s]
Traceback (most recent call last):
  File "/nas/ucb/mason/ethically-compliant-rl/CVPO/train_cvpo.py", line 246, in <module>
    train()
  File "/nas/ucb/mason/ethically-compliant-rl/.venv/lib/python3.9/site-packages/pyrallis/argparsing.py", line 158, in wrapper_inner
    response = fn(cfg, *args, **kwargs)
  File "/nas/ucb/mason/ethically-compliant-rl/CVPO/train_cvpo.py", line 211, in train
    agent.learn(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/agent/base_agent.py", line 236, in learn
    for epoch, _epoch_stat, info in trainer:
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/base_trainer.py", line 208, in __next__
    self.policy_update_fn(stats_train)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/offpolicy.py", line 106, in policy_update_fn
    self.policy.update(self.batch_size, self.train_collector.buffer)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 361, in update
    batch = self.process_fn(batch, buffer, indices)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/cvpo.py", line 216, in process_fn
    batch = self.compute_nstep_returns(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 494, in compute_nstep_returns
    metrics = self.get_metrics(buffer)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 392, in get_metrics
    metrics = [batch.rew, cost[:, 0], cost[:, 1]]
IndexError: index 1 is out of bounds for axis 1 with size 1
Traceback (most recent call last):
  File "/nas/ucb/mason/ethically-compliant-rl/CVPO/train_cvpo.py", line 246, in <module>
    train()
  File "/nas/ucb/mason/ethically-compliant-rl/.venv/lib/python3.9/site-packages/pyrallis/argparsing.py", line 158, in wrapper_inner
    response = fn(cfg, *args, **kwargs)
  File "/nas/ucb/mason/ethically-compliant-rl/CVPO/train_cvpo.py", line 211, in train
    agent.learn(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/agent/base_agent.py", line 236, in learn
    for epoch, _epoch_stat, info in trainer:
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/base_trainer.py", line 208, in __next__
    self.policy_update_fn(stats_train)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/offpolicy.py", line 106, in policy_update_fn
    self.policy.update(self.batch_size, self.train_collector.buffer)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 361, in update
    batch = self.process_fn(batch, buffer, indices)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/cvpo.py", line 216, in process_fn
    batch = self.compute_nstep_returns(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 494, in compute_nstep_returns
    metrics = self.get_metrics(buffer)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 392, in get_metrics
    metrics = [batch.rew, cost[:, 0], cost[:, 1]]
IndexError: index 1 is out of bounds for axis 1 with size 1
wandb: Waiting for W&B process to finish... (failed 1). Press Control-C to abort syncing.
wandb: üöÄ View run faithful-sweep-12 at: https://wandb.ai/ecrl/fast-safe-rl/runs/y463p9fp
wandb: Synced 5 W&B file(s), 0 media file(s), 0 artifact file(s) and 0 other file(s)
wandb: Find logs at: ./wandb/run-20230823_160103-y463p9fp/logs
wandb: üöÄ View run visionary-sweep-13 at: https://wandb.ai/ecrl/fast-safe-rl/runs/rt4v81cz
wandb: Synced 5 W&B file(s), 0 media file(s), 0 artifact file(s) and 0 other file(s)
wandb: Find logs at: ./wandb/run-20230823_230113-rt4v81cz/logs
2023-08-23 16:02:05,463 - wandb.wandb_agent - INFO - Cleaning up finished run: y463p9fp
2023-08-23 16:02:05,931 - wandb.wandb_agent - INFO - Agent received command: run
2023-08-23 16:02:05,931 - wandb.wandb_agent - INFO - Agent starting run with config:
	actor_lr: 0.0005
	critic_lr: 0.0005
	gamma: 0.095
2023-08-23 16:02:05,933 - wandb.wandb_agent - INFO - About to run command: /usr/bin/env python /nas/ucb/mason/ethically-compliant-rl/CVPO/train_cvpo.py --actor_lr=0.0005 --critic_lr=0.0005 --gamma=0.095
2023-08-23 23:02:06,073 - wandb.wandb_agent - ERROR - Detected 5 failed runs in a row, shutting down.
2023-08-23 23:02:06,073 - wandb.wandb_agent - INFO - To change this value set WANDB_AGENT_MAX_INITIAL_FAILURES=val
wandb: Terminating and syncing runs. Press ctrl-c to kill.
wandb: Currently logged in as: mason-nakamura1. Use `wandb login --relogin` to force relogin
wandb: WARNING Ignored wandb.init() arg project when running a sweep.
wandb: WARNING Ignored wandb.init() arg entity when running a sweep.
wandb: wandb version 0.15.8 is available!  To upgrade, please run:
wandb:  $ pip install wandb --upgrade
wandb: Tracking run with wandb version 0.14.2
wandb: Run data is saved locally in /nas/ucb/mason/ethically-compliant-rl/wandb/run-20230823_160209-0vc7t7u0
wandb: Run `wandb offline` to turn off syncing.
wandb: Syncing run light-sweep-15
wandb: ‚≠êÔ∏è View project at https://wandb.ai/ecrl/fast-safe-rl
wandb: üßπ View sweep at https://wandb.ai/ecrl/fast-safe-rl/sweeps/3qe1o66j
wandb: üöÄ View run at https://wandb.ai/ecrl/fast-safe-rl/runs/0vc7t7u0
2023-08-23 16:02:10,940 - wandb.wandb_agent - INFO - Running runs: ['0vc7t7u0']
[32;1mLogging data to logs/CVPO-sweep-700epochs/parking-v0-cost-[5.0]/cvpo_constraint_typelines_cost5.0_critic_lr0.0005_gamma0.095_step_per_epoch1000-70d8/progress.txt[0m
Epoch #1:   0%|          | 0/1000 [00:00<?, ?it/s]Epoch #1: 10046it [00:41, 240.77it/s]             Epoch #1: 10046it [00:41, 240.75it/s]
Traceback (most recent call last):
  File "/nas/ucb/mason/ethically-compliant-rl/CVPO/train_cvpo.py", line 246, in <module>
    train()
  File "/nas/ucb/mason/ethically-compliant-rl/.venv/lib/python3.9/site-packages/pyrallis/argparsing.py", line 158, in wrapper_inner
    response = fn(cfg, *args, **kwargs)
  File "/nas/ucb/mason/ethically-compliant-rl/CVPO/train_cvpo.py", line 211, in train
    agent.learn(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/agent/base_agent.py", line 236, in learn
    for epoch, _epoch_stat, info in trainer:
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/base_trainer.py", line 208, in __next__
    self.policy_update_fn(stats_train)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/offpolicy.py", line 106, in policy_update_fn
    self.policy.update(self.batch_size, self.train_collector.buffer)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 361, in update
    batch = self.process_fn(batch, buffer, indices)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/cvpo.py", line 216, in process_fn
    batch = self.compute_nstep_returns(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 494, in compute_nstep_returns
    metrics = self.get_metrics(buffer)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 392, in get_metrics
    metrics = [batch.rew, cost[:, 0], cost[:, 1]]
IndexError: index 1 is out of bounds for axis 1 with size 1
Traceback (most recent call last):
  File "/nas/ucb/mason/ethically-compliant-rl/CVPO/train_cvpo.py", line 246, in <module>
    train()
  File "/nas/ucb/mason/ethically-compliant-rl/.venv/lib/python3.9/site-packages/pyrallis/argparsing.py", line 158, in wrapper_inner
    response = fn(cfg, *args, **kwargs)
  File "/nas/ucb/mason/ethically-compliant-rl/CVPO/train_cvpo.py", line 211, in train
    agent.learn(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/agent/base_agent.py", line 236, in learn
    for epoch, _epoch_stat, info in trainer:
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/base_trainer.py", line 208, in __next__
    self.policy_update_fn(stats_train)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/offpolicy.py", line 106, in policy_update_fn
    self.policy.update(self.batch_size, self.train_collector.buffer)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 361, in update
    batch = self.process_fn(batch, buffer, indices)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/cvpo.py", line 216, in process_fn
    batch = self.compute_nstep_returns(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 494, in compute_nstep_returns
    metrics = self.get_metrics(buffer)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 392, in get_metrics
    metrics = [batch.rew, cost[:, 0], cost[:, 1]]
IndexError: index 1 is out of bounds for axis 1 with size 1
wandb: Waiting for W&B process to finish... (failed 1). Press Control-C to abort syncing.
Epoch #1:   0%|          | 0/1000 [00:00<?, ?it/s]Epoch #1: 13022it [00:50, 258.99it/s]             Epoch #1: 13022it [00:50, 258.96it/s]
Traceback (most recent call last):
  File "/nas/ucb/mason/ethically-compliant-rl/CPO/train_cpo.py", line 237, in <module>
    train()
  File "/nas/ucb/mason/ethically-compliant-rl/.venv/lib/python3.9/site-packages/pyrallis/argparsing.py", line 158, in wrapper_inner
    response = fn(cfg, *args, **kwargs)
  File "/nas/ucb/mason/ethically-compliant-rl/CPO/train_cpo.py", line 202, in train
    agent.learn(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/agent/cpo_agent.py", line 229, in learn
    return super().learn(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/agent/base_agent.py", line 353, in learn
    for epoch, _epoch_stat, info in trainer:
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/base_trainer.py", line 208, in __next__
    self.policy_update_fn(stats_train)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/onpolicy.py", line 104, in policy_update_fn
    self.policy.update(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 361, in update
    batch = self.process_fn(batch, buffer, indices)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/cpo.py", line 130, in process_fn
    batch = self.compute_gae_returns(batch, buffer, indices, self._lambda)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 421, in compute_gae_returns
    metrics = self.get_metrics(batch)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 392, in get_metrics
    metrics = [batch.rew, cost[:, 0], cost[:, 1]]
IndexError: index 1 is out of bounds for axis 1 with size 1
wandb: Waiting for W&B process to finish... (failed 1). Press Control-C to abort syncing.
wandb: üöÄ View run good-sweep-14 at: https://wandb.ai/ecrl/fast-safe-rl/runs/fsxb61zg
wandb: Synced 5 W&B file(s), 0 media file(s), 0 artifact file(s) and 0 other file(s)
wandb: Find logs at: ./wandb/run-20230823_160120-fsxb61zg/logs
Epoch #1:   0%|          | 0/1000 [00:00<?, ?it/s]Epoch #1: 10771it [00:42, 251.02it/s]             Epoch #1: 10771it [00:42, 250.99it/s]
Traceback (most recent call last):
  File "/nas/ucb/mason/ethically-compliant-rl/CPO/train_cpo.py", line 237, in <module>
    train()
  File "/nas/ucb/mason/ethically-compliant-rl/.venv/lib/python3.9/site-packages/pyrallis/argparsing.py", line 158, in wrapper_inner
    response = fn(cfg, *args, **kwargs)
  File "/nas/ucb/mason/ethically-compliant-rl/CPO/train_cpo.py", line 202, in train
    agent.learn(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/agent/cpo_agent.py", line 229, in learn
    return super().learn(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/agent/base_agent.py", line 353, in learn
    for epoch, _epoch_stat, info in trainer:
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/base_trainer.py", line 208, in __next__
    self.policy_update_fn(stats_train)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/onpolicy.py", line 104, in policy_update_fn
    self.policy.update(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 361, in update
    batch = self.process_fn(batch, buffer, indices)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/cpo.py", line 130, in process_fn
    batch = self.compute_gae_returns(batch, buffer, indices, self._lambda)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 421, in compute_gae_returns
    metrics = self.get_metrics(batch)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 392, in get_metrics
    metrics = [batch.rew, cost[:, 0], cost[:, 1]]
IndexError: index 1 is out of bounds for axis 1 with size 1
wandb: Waiting for W&B process to finish... (failed 1). Press Control-C to abort syncing.
wandb: - 0.027 MB of 0.027 MB uploaded (0.000 MB deduped)wandb: \ 0.027 MB of 0.029 MB uploaded (0.000 MB deduped)wandb: | 0.031 MB of 0.031 MB uploaded (0.000 MB deduped)wandb: üöÄ View run golden-sweep-14 at: https://wandb.ai/ecrl/fast-safe-rl/runs/of9m5b26
wandb: Synced 5 W&B file(s), 0 media file(s), 0 artifact file(s) and 0 other file(s)
wandb: Find logs at: ./wandb/run-20230823_160117-of9m5b26/logs
wandb: - 0.027 MB of 0.027 MB uploaded (0.000 MB deduped)wandb: \ 0.027 MB of 0.029 MB uploaded (0.000 MB deduped)wandb: üöÄ View run charmed-sweep-15 at: https://wandb.ai/ecrl/fast-safe-rl/runs/1obdrl1e
wandb: Synced 5 W&B file(s), 0 media file(s), 0 artifact file(s) and 0 other file(s)
wandb: Find logs at: ./wandb/run-20230823_160127-1obdrl1e/logs
2023-08-23 16:02:22,458 - wandb.wandb_agent - ERROR - Detected 5 failed runs in a row, shutting down.
2023-08-23 16:02:22,459 - wandb.wandb_agent - INFO - To change this value set WANDB_AGENT_MAX_INITIAL_FAILURES=val
wandb: Terminating and syncing runs. Press ctrl-c to kill.
2023-08-23 16:02:25,813 - wandb.wandb_agent - ERROR - Detected 5 failed runs in a row, shutting down.
2023-08-23 16:02:25,814 - wandb.wandb_agent - INFO - To change this value set WANDB_AGENT_MAX_INITIAL_FAILURES=val
wandb: Terminating and syncing runs. Press ctrl-c to kill.
2023-08-23 16:02:26,195 - wandb.wandb_agent - ERROR - Detected 5 failed runs in a row, shutting down.
2023-08-23 16:02:26,195 - wandb.wandb_agent - INFO - To change this value set WANDB_AGENT_MAX_INITIAL_FAILURES=val
wandb: Terminating and syncing runs. Press ctrl-c to kill.
Epoch #1:   0%|          | 0/1000 [00:00<?, ?it/s]Epoch #1: 11455it [00:54, 209.80it/s]             Epoch #1: 11455it [00:54, 209.79it/s]
Traceback (most recent call last):
  File "/nas/ucb/mason/ethically-compliant-rl/CVPO/train_cvpo.py", line 246, in <module>
    train()
  File "/nas/ucb/mason/ethically-compliant-rl/.venv/lib/python3.9/site-packages/pyrallis/argparsing.py", line 158, in wrapper_inner
    response = fn(cfg, *args, **kwargs)
  File "/nas/ucb/mason/ethically-compliant-rl/CVPO/train_cvpo.py", line 211, in train
    agent.learn(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/agent/base_agent.py", line 236, in learn
    for epoch, _epoch_stat, info in trainer:
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/base_trainer.py", line 208, in __next__
    self.policy_update_fn(stats_train)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/offpolicy.py", line 106, in policy_update_fn
    self.policy.update(self.batch_size, self.train_collector.buffer)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 361, in update
    batch = self.process_fn(batch, buffer, indices)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/cvpo.py", line 216, in process_fn
    batch = self.compute_nstep_returns(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 494, in compute_nstep_returns
    metrics = self.get_metrics(buffer)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 392, in get_metrics
    metrics = [batch.rew, cost[:, 0], cost[:, 1]]
IndexError: index 1 is out of bounds for axis 1 with size 1
Traceback (most recent call last):
  File "/nas/ucb/mason/ethically-compliant-rl/CVPO/train_cvpo.py", line 246, in <module>
    train()
  File "/nas/ucb/mason/ethically-compliant-rl/.venv/lib/python3.9/site-packages/pyrallis/argparsing.py", line 158, in wrapper_inner
    response = fn(cfg, *args, **kwargs)
  File "/nas/ucb/mason/ethically-compliant-rl/CVPO/train_cvpo.py", line 211, in train
    agent.learn(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/agent/base_agent.py", line 236, in learn
    for epoch, _epoch_stat, info in trainer:
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/base_trainer.py", line 208, in __next__
    self.policy_update_fn(stats_train)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/trainer/offpolicy.py", line 106, in policy_update_fn
    self.policy.update(self.batch_size, self.train_collector.buffer)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 361, in update
    batch = self.process_fn(batch, buffer, indices)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/cvpo.py", line 216, in process_fn
    batch = self.compute_nstep_returns(
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 494, in compute_nstep_returns
    metrics = self.get_metrics(buffer)
  File "/nas/ucb/mason/ethically-compliant-rl/FSRL/fsrl/policy/base_policy.py", line 392, in get_metrics
    metrics = [batch.rew, cost[:, 0], cost[:, 1]]
IndexError: index 1 is out of bounds for axis 1 with size 1
wandb: Waiting for W&B process to finish... (failed 1). Press Control-C to abort syncing.
wandb: üöÄ View run light-sweep-15 at: https://wandb.ai/ecrl/fast-safe-rl/runs/0vc7t7u0
wandb: Synced 5 W&B file(s), 0 media file(s), 0 artifact file(s) and 0 other file(s)
wandb: Find logs at: ./wandb/run-20230823_160209-0vc7t7u0/logs
2023-08-23 16:03:22,842 - wandb.wandb_agent - ERROR - Detected 5 failed runs in a row, shutting down.
2023-08-23 16:03:22,843 - wandb.wandb_agent - INFO - To change this value set WANDB_AGENT_MAX_INITIAL_FAILURES=val
wandb: Terminating and syncing runs. Press ctrl-c to kill.
